<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>A spatio-temporal framework for modelling wastewater concentration during the COVID-19 pandemic</title>
    <meta charset="utf-8" />
    <meta name="author" content="Marta Blangiardo" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/panelset/panelset.css" rel="stylesheet" />
    <script src="libs/panelset/panelset.js"></script>
    <!-- (Re)Defines a bunch of LaTeX commands that can then be used directly in the .Rmd file as '\command{...}' -->
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      TeX: {
        /* This enables color macros */
        extensions: ["color.js"],
        Macros: {
          /* Probability & mathematical symbols */
          Pr: "{\\style{font-family:inherit; font-size: 110%;}{\\text{Pr}}}",
          exp: "{\\style{font-family:inherit; font-size: 105%;}{\\text{exp}}}",
          log: "{\\style{font-family:inherit; font-size: 105%;}{\\text{log}}}",
          ln: "{\\style{font-family:inherit; font-size: 105%;}{\\text{ln}}}",
          logit: "{\\style{font-family:inherit; font-size: 100%;}{\\text{logit}}}",
          HR: "{\\style{font-family:inherit; font-size: 105%;}{\\text{HR}}}",
          OR: "{\\style{font-family:inherit; font-size: 105%;}{\\text{OR}}}",
          E: "{\\style{font-family:inherit; font-size: 105%;}{\\text{E}}}",
          Var: "{\\style{font-family:inherit; font-size: 105%;}{\\text{Var}}}",
          Cov: "{\\style{font-family:inherit; font-size: 105%;}{\\text{Cov}}}",
          Corr: "{\\style{font-family:inherit; font-size: 105%;}{\\text{Corr}}}",
          DIC: "{\\style{font-family:inherit; font-size: 105%;}{\\text{DIC}}}",
          se: "{\\style{font-family:inherit; font-size: 100%;}{\\text{se}}}",
          sd: "{\\style{font-family:inherit; font-size: 100%;}{\\text{sd}}}",
          kld: "{\\style{font-family:inherit; font-size: 100%;}{\\text{kld}}}",
          /* Distributions */
          dnorm: "{\\style{font-family:inherit;}{\\text{Normal}}}",
          dt: "{\\style{font-family:inherit;}{\\text{t}}}",
          ddirch: "{\\style{font-family:inherit;}{\\text{Dirichlet}}}",
          dmulti: "{\\style{font-family:inherit;}{\\text{Multinomial}}}",
          dbeta: "{\\style{font-family:inherit;}{\\text{Beta}}}",
          dgamma: "{\\style{font-family:inherit;}{\\text{Gamma}}}",
          dbern: "{\\style{font-family:inherit;}{\\text{Bernoulli}}}",
          dbin: "{\\style{font-family:inherit;}{\\text{Binomial}}}",
          dpois: "{\\style{font-family:inherit;}{\\text{Poisson}}}",
          dweib: "{\\style{font-family:inherit;}{\\text{Weibull}}}",
          dexp: "{\\style{font-family:inherit;}{\\text{Exponential}}}",
          dlnorm: "{\\style{font-family:inherit;}{\\text{logNormal}}}",
          dunif: "{\\style{font-family:inherit;}{\\text{Uniform}}}",
          /* LaTeX formatting */
          bm: ["{\\boldsymbol #1}",1],
          /* These create macros to typeset numbers in maths with the basic font */
          0: "{\\style{font-family:inherit; font-size: 105%;}{\\text{0}}}",
          1: "{\\style{font-family:inherit; font-size: 105%;}{\\text{1}}}",
          2: "{\\style{font-family:inherit; font-size: 105%;}{\\text{2}}}",
          3: "{\\style{font-family:inherit; font-size: 105%;}{\\text{3}}}",
          4: "{\\style{font-family:inherit; font-size: 105%;}{\\text{4}}}",
          5: "{\\style{font-family:inherit; font-size: 105%;}{\\text{5}}}",
          6: "{\\style{font-family:inherit; font-size: 105%;}{\\text{6}}}",
          7: "{\\style{font-family:inherit; font-size: 105%;}{\\text{7}}}",
          8: "{\\style{font-family:inherit; font-size: 105%;}{\\text{8}}}",
          9: "{\\style{font-family:inherit; font-size: 105%;}{\\text{9}}}",
          /* Health economics quantities */
          icer: "{\\style{font-family:inherit; font-size: 100%;}{\\text{ICER}}}",
          nb: "{\\style{font-family:inherit; font-size: 100%;}{\\text{NB}}}",
          ol: "{\\style{font-family:inherit; font-size: 100%;}{\\text{OL}}}",
          ceac: "{\\style{font-family:inherit; font-size: 100%;}{\\text{CEAC}}}",
          evpi: "{\\style{font-family:inherit; font-size: 100%;}{\\text{EVPI}}}",
          evppi: "{\\style{font-family:inherit; font-size: 100%;}{\\text{EVPPI}}}",
          evsi: "{\\style{font-family:inherit; font-size: 100%;}{\\text{EVSI}}}"
        }
      }
    });
    </script>
    <link rel="stylesheet" href="assets/beamer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">




class: title-slide

# A spatio-temporal framework for modelling wastewater concentration during the COVID-19 pandemic&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt; 

## Marta Blangiardo

### [MRC Centre for Environment and Health](https://environment-health.ac.uk/) | Imperial College London    

### Workshop on Ecological and Environmental Statistics, Lancaster University 

&lt;!-- Can also separate the various components of the extra argument 'params', eg as in 
### Workshop on Ecological and Environmental Statistics, Lancaster University, 12th September 2023, A spatio-temporal framework to integrate wastewater and disease prevalence for public health surveillance
--&gt;

12th September 2023

---

layout: true  

.my-footer[ 
.alignleft[ 
&amp;nbsp; &amp;copy; Marta Blangiardo
]
.aligncenter[
A spatio-temporal framework to integrate wastewater and disease prevalence for public health surveillance 
]
.alignright[
Workshop on Ecological and Environmental Statistics, NA 
]
] 


---

&lt;!-- I have known Sylvia for many years, since 2005, actually 2004 when i met her at a conference as a PhD student and you know when you meet your scientific hero, read the papers, very close to ask her for an autograph. Then I moved to London as she asked me to apply for a postdoc and I was "really, me?", well I did, I got the job after an interview over the phone when I was petrified of not understanding the questions (no zoom back then) and I move to London. I really love Sylvia, she has been a great, supportive boss, a wonderful friend, we shared a lot over the years, and I am really lucky that I met her and so happy to celebrate her today with you all. --&gt;

&lt;!--&lt;center&gt;&lt;img src=./img/SylviaMarta2.jpeg width='30%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

.pull-left[&lt;center&gt;&lt;img src=./img/SylviaMarta.jpeg width='50%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

.pull-right[&lt;center&gt;&lt;img src=./img/Office2012.jpg width='40%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;]

--&gt;


# The Lab

.pull-left[&lt;center&gt;&lt;img src=./img/Turing.png width='110%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

.center[[Turing-RSS Health Data lab](https://www.turing.ac.uk/research/research-projects/turing-rss-health-data-lab)]

&lt;span style="display:block; margin-top: 50px ;"&gt;&lt;/span&gt;

.small[
- Bayesian imputation of COVID-19 positive test counts for nowcasting under reporting lag
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
- Spatial and temporal modelling of incidence and prevalence of COVID-19
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
- Estimating COVID-19 prevalence and transmission from multiple sources: de-biasing Pillar 2 data  
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
- Interoperability of Statistical Models in Pandemic Preparedness: Principles and Reality
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
- .red[A spatio-temporal framework for modelling wastewater concentration during the COVID-19 pandemic]
]
]

.pull-right[
&lt;center&gt;&lt;img src=./img/Lab.png width='90%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

---
  
#Wastewater-based epidemiology

&lt;span style="display:block; margin-top: 50px ;"&gt;&lt;/span&gt;
&lt;center&gt;
.content-box-green[.center[.large[A collection of tools
and methods for surveillance and monitoring disease outbreaks using bio-
chemical analysis of wastewater samples as the primary outcome measure]]]
&lt;/center&gt;

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Wastewater has been used over the years as an epidemiological tool
  - no selection bias
  - cheap

- Wastewater based epidemiology started as a way to use wastewater to estimate exposure to chemicals in the population

  - Illicit drugs, psychoactive substances
  - Pesticides and other endocrine disruptors
  - Bacteria, viruses, fungi
  
Success stories:

  - retrospective prediction of outbreaks like hepatitis A and norovirus
  - polio - WHO guidelines for monitoring polio includes wastewater
  

---

# How does it work 

&lt;span style="display:block; margin-top: 50px ;"&gt;&lt;/span&gt;

&lt;center&gt;&lt;img src=./img/WW.png width='100%' title=''&gt;&lt;/center&gt;

---


# Wastewater during the COVID-19 pandemic

- WBE has been recognised as an economically efficient approach for disease surveillance (Manuel et al., 2022)

- Methods to detect the presence of SARS-CoV-2 RNA in wastewater have
been developed in a number of countries (Tlhagale et al., 2022)

- Scientific contributions to date have focused on the level of the sewage treatment works (STWs) where the
measurements of SARS-CoV-2 RNA from wastewater are obtained.

- No attempt has previously been made to develop a
spatially resolved model to estimate wastewater viral
concentration over a spatially continuous domain.

&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
--

## Aim

- Develop a statistical model to estimate viral concentration in wastewater at fine spatio-temporal resolutions covering the whole of England

&lt;!-- - Collaboration between UKHSA, RSS, Alan Turing Institute --&gt;


---

# Modelling spatial data

- **Point-referenced data**: the exact location of the occurrences is known
  -  can be collected through specialized survey / monitoring network...
  - if locations are **fixed** (monitoring stations, postcodes in an area) and the variable of interest is measured at each location (e.g. presence/absence of cases, pollution concentrations)
  
  `\(\Rightarrow\)` .red[geostatistical framework.]


&lt;!-- .center[.content-box-blue[
**First law of geography** (W. Tobler, 1970): everything is related to everything else, but near things are more related than distant things]
]

--&gt;

--

- We want to .red[reconstruct a latent spatial (temporal) surface] `\(\class{red}{\mathbf{S}}\)` .red[from a finite set of noisy observations and their spatial (temporal) location.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Use the spatial dependence to **predict** values of the spatial
surface (together with associated uncertainty) at locations where
there are no observations.



---

# Gaussian fields

- A spatial process is a Gaussian field if `\(\left(y(\bm s_1),\ldots, y(\bm s_n)\right)\)` follows a multivariate Normal distribution with mean `\(\bm \mu=\left(\mu(\bm s_{1}), \ldots, \mu(\bm s_{n})\right)\)` and spatially structured covariance matrix `\(\bm \Sigma\)`. 

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- The spatial process is called .red[second-order stationary] if 
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
(i) `\(\bm \mu\)` is constant (i.e. `\(\mu(\bm s_{i}) = \mu\)` for each `\(i\)`) 
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
(ii) the spatial covariance function depends only on the distance vector `\((\bm s_{i}-\bm s_{j})\)`. 

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- A stationary process is .red[isotropic] if the covariance does not depend on the direction but just on the Euclidean distance  `\(||\bm s_{i}-\bm s_{j}|| \in \mathbb{R}\)`.

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Several functions are available for the spatial covariance function (eg exponential, Mat\'ern, spherical, etc.) parameterized by some parameters (eg spatial variance, range, etc.).

---



# Model for geostatistical data
Data are measured (possibly with error) at `\(n\)` spatial locations `\((\bm s_{1},\ldots,\bm s_{n})\)` and are denoted by `\(\bm y=\left(y(\bm s_1),\ldots, y(\bm s_n)\right)=(y_1,\ldots,y_n)\)`.

Usually the following model is assumed

`\begin{align*}
y_s &amp;\sim N(\mu_s, \sigma^2_y)\\
\mu_s &amp;= \alpha + \bm{x}_s\boldsymbol{\beta} + \xi_s
\end{align*}`

--

 where

- `\(\bm{x}\boldsymbol{\beta}\)`  is the large scale component, defined by some covariates.
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
- `\(\boldsymbol{\xi}\)`  is a zero mean Gaussian spatial process commonly assumed to be stationary and isotropic with covariance function `\(Cov(\xi_s,\xi_j)\)`  which depends only on the distance between the locations.
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
- `\(\sigma^2_y\)`  represents the measurement error variance (known as nugget effect).
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
- `\(Cov(\xi_s,\xi_j)\)` is a function of the range, which tells us how much the correlation decays with the distance

---

# Mat&amp;eacute;rn covariance function

`$$\text{Cov}(\xi(\bm s_i),\xi(\bm s_j))= \text{Cov}(\xi_i,\xi_j)=\frac{\sigma^2}{\Gamma(\lambda)2^{\lambda-1}}\left(\kappa ||\bm s_i - \bm s_j||\right)^\lambda K_\lambda\left(\kappa ||\bm s_i - \bm s_j||\right)$$`
where
-  `\(||\bm s_i - \bm s_j||\)`  is the Euclidean distance between two generic locations `\(\bm s_i, \bm s_j \in \mathbb{R}^d\)`
- `\(\sigma^2\)`  is the .red[variance]
- `\(K_\lambda\)`  denotes the modified Bessel function of second kind and order `\(\lambda&gt;0\)`, which measures the degree of .red[smoothness] of the process. 
- `\(\kappa&gt;0\)`  is a .red[scale] parameter related to the range `\(r\)`, i.e. the distance at which the spatial correlation becomes almost null. 
- Typically, the empirically derived definition for the range is `\(r=\frac{\sqrt{8 \lambda}}{\kappa}\)`, with `\(r\)`  corresponding to the distance at which the spatial correlation is close to 0.1, for each `\(\lambda\geq 1/2\)`.


---


# Possible approaches for estimation
- .red[Classical geostatistical approach]: the empirical variogram is used as an exploratory tool and the mean and the covariance function parameters are estimated through  least square methods usually adopting a two step procedure

`\(\rightarrow\)`  very simple and not computationally intensive;

`\(\rightarrow\)`  two-step approach to prediction;

`\(\rightarrow\)`  difficult to account for uncertainty. 

- .red[(Bayesian) Model-based  approach]:  hierarchical model specification and posterior predictive distribution for prediction;

`\(\rightarrow\)`  easy to allow for uncertainty;

`\(\rightarrow\)`  flexible in accounting for any type of distribution;

`\(\rightarrow\)` high computationally costs.


---

# Data

- SARS-CoV-2 viral concentrations obtained from the 303 Sewage Treatment Works (STWs) for which the data were publicly available from the Environmental Monitoring for Health Protection (EMHP) wastewater surveillance programme.


.pull-left[
&lt;center&gt;&lt;img src=./img/PopCoverage.png width='100%' title='% pop coverage'&gt;&lt;/center&gt;
.center[Percentage population coverage]
]

.pull-right[
&lt;center&gt;&lt;img src=./img/STW.png width='61%' title='Location of STWs and their catchment areas'&gt;&lt;/center&gt;
.center[Location of STWs and their catchment areas]
]

---

# Data

- At each site three to four samples were collected per week between 1 June 2021 to 30 March 2022. 

--
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- We consider the weekly average of flow-normalised viral concentration measurements reported as the number of SARS-CoV-2 N1 gene copies per litre of wastewater (gc/L), obtained through reverse transcriptase quantitative polymerase chain reaction (RT-qPCR) analysis of the wastewater samples.

--
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- We also include socio-demographic, land use covariates and wastewater genomic data:

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

  1. Index of Multiple Deprivation (2019)
  2. Black, Asian and Minority Ethnic (BAME) proportion (Census, 2011)
  3. Land cover (Copernicus, 2018)
  4. Population density (ONS, 2019)
  5. Age structure (ONS, 2019)
  6. Percentage coverage of the SARS-CoV-2 genome in the sample
  7. Single-Nucleotide Polymorphism (SNP) data in the sample

---

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  u_i + v_t + z_{it}$$`
 
---

count:false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} ={\color{red}{\alpha_{region_{g_i}}}} + \bm{x}_i \bm{\beta} +  u_i + v_t + z_{it}$$`

- `\(\color{red}{\alpha_{region_{g_i}}}\)`: region specific intercept, modelled as exchangeable

---

count: false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} = \alpha_{region_{g_i}} + {\color{red}\bm{x}_i \bm{\beta}} +  u_i + v_t + z_{it}$$`

- `\(\color{red}{\bm{x}_i \bm{\beta}}\)`: vector of covariates and their effects

---

count: false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  {\color{red}u_i} + v_t + z_{it}$$`

- `\(\color{red}{u_i}\)`: spatial random effect with an exchangeable prior 

---


count: false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  u_i + {\color{red}v_t} + z_{it}$$`

- `\(\color{red}{v_t}\)`: temporal random effect modelled with first order random walk

---

# Spatio-temporal random effect

`$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  u_i + v_t + {\color{red}z_{it}}$$`

- `\(\color{red}{z_{it}}\)`: spatio-temporal random effect, modelled as a temporal autoregressive structure with spatially correlated innovations, similar to Cameletti et al. (2011):
`\begin{align*}
t = 1 \qquad \boldsymbol{z}_t &amp;\sim \text{Normal}(0,\Sigma_{z})\\
t &gt; 1 \qquad \boldsymbol{z}_t &amp;\sim \text{Normal}(\rho\boldsymbol{z}_{t-1},\Sigma_{z})
\end{align*}`
with `\(\rho\)` as the temporal autoregressive coefficient. 

--

The  `\(\Sigma_{z}\)` matrix represents the spatial structure and is defined using a Mat&amp;eacute;rn covariance function.

&lt;!--$$\text{Cov}(\bm{z}_i,\bm{z}_s)\ = \frac{\sigma^2_{z}}{\Gamma(\lambda)2^{\lambda-1}}(\kappa ||\bm{z}_i - \bm{z}_s||)^{\lambda}K_{\lambda}(\kappa ||\bm{z}_i - \bm{z}_s||)$$

where `\(\Gamma(\lambda)\)` is the gamma function, `\(K_{\lambda}\)` is the modified Bessel function of second kind with order `\(\lambda\)`. The parameter `\(\kappa&gt;0\)` controls the range `\(\rho=\sqrt{8\lambda}/\kappa\)` and indicate the distance where the spatial correlation is close to 0.1. 

--&gt;

---

# Spatial prediction

- We consider the population-weighted centroids of all the 32,844 Lower Super Output Areas (LSOAs) in England, these being the smallest areas 
for which all the covariates are available. 

Given observed data `\(\boldsymbol{y}\)` we sample `\(\boldsymbol{y}^\prime\)`, the viral concentrations at the LSOA centroids over 44 weeks, from the posterior predictive distribution:
`\begin{align}
\pi(\boldsymbol{y}^\prime \mid \boldsymbol{y}) &amp;= \int p(\boldsymbol{y}^\prime \mid \boldsymbol{\theta}) p(\boldsymbol{\theta} \mid \boldsymbol{y}) d\boldsymbol{\theta}\\
\boldsymbol{\theta} &amp;= (\alpha, \boldsymbol{\beta}, \boldsymbol{v}, \boldsymbol{g}, \boldsymbol{u}, \boldsymbol{z}, \sigma^2_y)\ .
\end{align}`


--

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;


The posterior predictive distribution at the LSOA centroids can be combined to return corresponding predictions at coarser geographical scales, as required. 

  - Here we consider all the Lower Tier Local Authorities in England and average the corresponding LSOA values, weighted by the population in each LSOA.


- Implemented in R-INLA 


---

# Spatio-temporal patterns in wastewater concentration

.pull-left[

Viral concentrations in wastewater varied substantially across England over the observation period:
&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
  - first sharp increase at the beginning of the study period
&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
  - second increase around November 2021 with a peak just after Christmas
&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
  - third increase in February-March 2022 
&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;  
  - London shows the highest overall viral concentration level, while the North-West and Yorkshire regions are characterised by the lowest values.
]

.pull-right[
&lt;center&gt;&lt;img src=./img/Fig2rescaled.jpg width='90%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]


&lt;!---
# Spatio-temporal patterns in wastewater concentration

.pull-left[

- The rise and fall of viral concentrations at the national level did not happen at the same time across all LTLAs
&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
  - At the beginning of June 2021, low values are estimated across most LTLAs while inner London shows higher values.
&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
  - By mid-July 2021, an increment is clearly visible across the country, but a high degree of variation is noticeable.
  - 
]

.pull-right[
Posterior means of the predicted wastewater concentration at LSOA level for selected weeks

&lt;center&gt;&lt;img src=./img/fig3rescaled.png width='70%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

--&gt;

---

# Detection of areas at increased risk of outbreaks

.panelset[
.panel[.panel-name[Detection]
We used the posterior predictive distribution to detect increases in the wastewater viral concentration. For each LTLA `\((l=1,\ldots,L)\)` an increase in viral concentration over two consecutive weeks is the event (E):
`\begin{eqnarray}
E = I(\mu_{lt} &gt; \mu_{l(t-1)} &gt; \mu_{l(t-2)})
\end{eqnarray}`
where `\(I(\cdot)\)` is the indicator function.

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
A high predictive probability, `\({\rm Prob}(E)&gt;c\)` for a pre-specified threshold `\(c\)`, would then trigger a warning of a potential outbreak. This approach has been used for public health surveillance in both high-income (Diggle et al., 2005) and low-to-middle-income (Diggle et al., 2007) countries. 
]

.panel[.panel-name[Mapping]

&lt;center&gt;&lt;img src=./img/Detection.png width='50%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]
]

---

#Interactive dashboard
To visualise these rich model outputs, we have created a dynamic and interactive dashboard

&lt;span style="display:block; margin-top: -50px ;"&gt;&lt;/span&gt;

&lt;iframe src="https://b-rowlingson.gitlab.io/wwatlas/" width="140%" height="100%" data-external="1" style="transform: scale(.8);border:none"&gt;&lt;/iframe&gt;


---

# Wastewater viral concentration and COVID-10 prevalence

- Several studies have evaluated the relationship between viral load in
wastewater and prevalence of COVID-19 disease (e.g. Shah et al. (2022), Proverbio et al. (2022))

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Morvan et al. (2022) analysed data from 45 STWs across England
between July 2020 and March 2021. Using a multilevel modelling approach
they reported that wastewater samples can be used successfully to predict
COVID-19 prevalence, with a lead time of four to five days. 

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- We evaluated *in a descriptive way* the accordance between the predicted wastewater concentration and COVID-19 debiased disease prevalence.


&lt;!-- # COVID-19 debiased prevalence

- Debiased prevalence was estimated combining testing data and the REal-time Assessment of Community Transmission (REACT) randomised survey Riley et al. (2020) using the method proposed by Nicholson et al. (2022)

- Test positivity rates in Pillar 2 data are strongly biased upwards relative to the population prevalence proportion as the testing is directed
at the higher risk population

- From the randomised survey we get the estimated number of infected `\(I\)`

- From the targeted tests we define:
`\begin{align*}
P(n \text{ of } N | I, \delta, \nu) &amp;= \text{Binomial}(n | I, P(\text{Tested} | \text{Infected})) \times \\
&amp; \text{Binomial}(N-n | M-I, P(\text{Tested} | \text{Not Infected}))
\end{align*}`

  - `\(N\)` tests and `\(n\)` positive ones.

  - `\(\delta = log\left(\text{Odds}(\text{Tested} | \text{Infected})/\text{Odds}(\text{Tested} | \text{Not Infected})\right)\)`  

  - `\(\nu=\log \text{Odds}(\text{Tested} | \text{Not Infected})\)`, easy to get from `\(\logit[(N − n)/M ]\)`

- Specifying a Beta prior on `\(P(\text{Tested} | \text{Infected})\)` and using a Gaussian approximation we can use some conjugacy property and obtain posterior estimates of `\(\delta\)`

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

- Extensive cross-validation performed
--&gt;

---

# Wastewater viral concentration and COVID-19 prevalence

- Debiased prevalence was estimated combining testing data and the REal-time Assessment of Community Transmission (REACT) randomised survey Riley et al. (2020) using the method proposed by Nicholson et al. (2022)


&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

.pull-left[
&lt;span style="display:block; margin-top: 60px ;"&gt;&lt;/span&gt;
Correspondence between changes in wastewater viral concentration and changes in debiased prevalence.
&lt;span style="display:block; margin-top: 60px ;"&gt;&lt;/span&gt;

Legend:
.small[Dark green = high probability of rise in both viral concentration and prevalence; 

Pale green = neither viral concentration nor prevalence exhibited high probability of rise; 

Dark pink = rise was detected in prevalence but not in viral concentration

Light pink = rise was detected in viral concentration but not in prevalence.
]
]

.pull-right[
 
&lt;center&gt;&lt;img src=./img/fig5_resize.jpg width='90%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]



---

# Wastewater viral concentration and COVID-19 prevalence

.pull-left[The relationship between viral concentration in
wastewater and COVID-19 prevalence appears to be .red[complex
and varying over both space and time.]]


.pull-right[&lt;center&gt;&lt;img src=./img/fig6.jpg width='50%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;]

---

# Wastewater in the post-pandemic era to monitor COVID

- Data on randomised surveys and testing become more sparse
&lt;span style="display:block; margin-top: -5px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` Provide information on coarse geographical resolution (e.g. regional or national)

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Wastewater can track prevalence well and is cheaper to measure
&lt;span style="display:block; margin-top: -5px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` A monitoring system needs to be maintained 


--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

What data do we have (in the context of England)?

&lt;!--a. Assuming we have wastewater viral concentrations regularly measured at STWs across the country

`\(\rightarrow\)` we can use the geostatistical model to predict the LTLA weekly concentration.

b. .red[LTLA weekly prevalence] estimates are only available up to a certain time point, but estimates at the weekly-coarse level remain available --&gt;


&lt;center&gt;&lt;img src=./img/WWCovid.png width='60%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

.content-box-green[Can wastewater data help disaggregate coarse level prevalence to
a finer spatial scale?
]

---

# Modelling framework

- Coarse (national) and local (LTLA) models jointly fitted

- Wastewater predictions available from the geostatistical model

- Posterior distribution of COVID-19 prevalence obtained through Nicholson et al. (2022) 

--

### Coarse (national)
`\begin{align*}
\tilde{P}_{t} &amp;\sim \mbox{N}(\tilde{\mu}_{t},\tilde{\sigma}^2_{t})\\
\tilde{\mu}_{t} &amp;= \alpha + b_t + d_t \tilde{y}^{\prime}_{t}
\end{align*}`

where 

- `\(\tilde{P}_{t}\)` and `\(\tilde{\sigma}^2_{t}\)` are posterior mean and variance of the national aggregation of the prevalence (population weighted)
- `\(\tilde{y}^{\prime}_{t}\)` is a similar aggregation for wastewater concentration

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- `\(b_t\)` is modelled using a RW1
- `\(d_t\)` is modelled using a RW2

---

# Modelling framework

- Coarse (national) and local (LTLA) models jointly fitted

- Wastewater predictions available from the geostatistical model

- Posterior distribution of COVID-19 prevalence obtained through Nicholson et al. (2022) 

--

### Local
`\begin{align*}
p_{lt} &amp;\sim \mbox{N}(\mu_{lt},\sigma^2_{lt})\\
\mu_{lt} &amp;= (\alpha + u_l) + (b_t + v_{lt}) + (d_t + m_l)y^{\prime}_{lt}\\
\bm{y}^{\prime}_{t} &amp;\sim \mbox{MVN}(\bm{g}_t, \bm{S}_t)
\end{align*}`

where 

- `\(p_{lt}\)` and `\(\sigma^2_{lt}\)` are posterior mean and variance of the COVID-19 prevalence 
- `\(\bm{g}_t\)` and `\(\bm{S}_t\)` are the posterior mean and covariance matrix from the wastewater model
- `\(u_{l}\)`, `\(v_{lt}, m_l\)` are modelled as exchangeable


---

# Implementation

- Both the local and national components are jointly fitted

- Alternative models: 

1\. No wastewater

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` temporal trend for the coarse level model `\(\tilde{\mu}_{t} = \alpha + b_t + d_t\)`
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` spatial and temporal trend for the local level model `\(\mu_{lt} = (\alpha + u_l) + (b_t + v_{lt})\)`

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;

2\. No temporal varying coefficients
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` linear relationship with wastewater `\(\tilde{\mu}_{t} = \alpha + b_t + d \tilde{y}^{\prime}_{t}\)`
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` spatial-varying coefficient for wastewater `\(\mu_{lt} = (\alpha + u_l) + (b_t + v_{lt}) + (d + m_l)y^{\prime}_{lt}\)`

--

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;

- *Study period*: 40 weeks

- **Training window**: first 20 weeks (.red[LTLA + national weekly prevalence] + .blue[LTLA + national weekly wastewater])

- **Disaggregation window**: subsequent 5/10/15/20 weeks (.red[national weekly prevalence] + .blue[LTLA + national weekly wastewater])

---

# Results
.panelset[
.panel[.panel-name[RMSE]

.pull-left[&lt;center&gt;&lt;img src=./img/RMSE_LTLA.png width='80%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

.pull-right[
&lt;center&gt;&lt;img src=./img/Boxplot.png width='130%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

&lt;center&gt;&lt;img src=./img/Results.png width='130%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

]

.panel[.panel-name[Disease prevalence comparison]

.pull-left[&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
&lt;center&gt;&lt;img src=./img/Heatmap.png width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

]


.pull-right[&lt;span style="display:block; margin-top: 5px ;"&gt;&lt;/span&gt;
&lt;center&gt;&lt;img src=./img/Scatterplot.png width='80%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]
- Some periods (e.g. Christmas are showing larger differences)

- Some areas are consistently underestimated/overestimated
 
]
]

&lt;!--&lt;span style="display:block; margin-top: -20px ;"&gt;&lt;/span&gt;
&lt;center&gt;&lt;img src=./img/Boxplot.png width='80%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
--&gt;

---

&lt;!--
# Results

.pull-left[&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
&lt;center&gt;&lt;img src=./img/Heatmap.png width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]


.pull-right[&lt;span style="display:block; margin-top: 5px ;"&gt;&lt;/span&gt;
&lt;center&gt;&lt;img src=./img/Scatterplot.png width='80%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

`\(\rightarrow\)` Some periods (e.g. Christmas are showing larger differences)

`\(\rightarrow\)` Some areas are consistently underestimated/overestimated

--&gt;

# What's next

How can wastewater be incorporated into a real-time, multiplex public health surveillance system?

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

&lt;center&gt;&lt;img src=./img/Surveillance_representation.png width='60%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

---

# What's next

How can wastewater be incorporated into a real-time, multiplex public health surveillance system?

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

&lt;center&gt;&lt;img src=./img/Surveillance_representation.png width='60%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

---

# Discussion

- Wastewater is yet an underexplored epidemiological resource 
  - Cheap, population-based, no selection bias.
  - We ran the first study to provide estimates of SARS-CoV-2 viral concentrations in wastewater on a spatially resolved domain, which can then be aggregated to any required spatial resolution to inform public health decisions.

--
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Viral load can be affected by several factors other than local disease prevalence, including meteorology, type of sewage system, and population characteristics which need to be taken into consideration.

--
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Daily data may be better suited to assessment of leads or lags between the signals from wastewater and from traditional epidemiological metrics. 
  - this would present additional modelling challenges to account for potential weekly periodicity, the irregular pattern of wastewater sampling days leading to missing values.


---

# Discussion

- Relationship with disease is not trivial. In a descriptive way we found a spatial and temporal dynamicity and evidence of non-linearity. It could depend on several factors like vaccination, variants and metereology.

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

The proposed framework for disaggregation demonstrates the added-value of wastewater data to inform local prevalence

`\(\rightarrow\)` Continuous running of randomised surveys at a reduced capacity to provide coarse prevalence estimates remains a key pillar in COVID monitoring, anchoring the disaggregation and the relationship

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- The wastewater-prevalence relationship seems stable in time over the study period but will likely need to be recalibrated over longer times

`\(\rightarrow\)` occasional running of randomised surveys at a fine spatial scale is required

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

`\(\rightarrow\)` impact on the results will need to be evaluated

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Starting point but a long way to go

`\(\rightarrow\)` should be extended to include other data sources, such as syndromic and secondary care data, to enable quantitatively accurate monitoring of disease evolution across a population.

---

# References

Cameletti, M. et al. (2011). "Comparing spatio-temporal models for
particulate matter in Piemonte". In: _Environmetrics_ 22.8, pp.
985-996.

Diggle, P. et al. (2005). "Point process methodology for on-line
spatio-temporal disease surveillance". In: _Environmetrics_ 16, pp.
423-434.

Diggle, P. et al. (2007). "Spatial modelling and prediction of Loa loa
risk: decision making under uncertainty". In: _Annals of Tropical
Medicine and Parasitology_ 101, pp. 499-509.

Manuel, D. et al. (2022). "Strengthening Public Health Surveillance
Through Wastewater Testing".

Morvan, M. et al. (2022). "An analysis of 45 large-scale wastewater
sites in England to estimate SARS-CoV-2 community prevalence". In:
_Nature Communications_ 13.4313.

Nicholson, G. et al. (2022). "Improving local prevalence estimates of
SARS-CoV-2 infections using a causal debiasing framework". In: _Nature
microbiology_ 7.1, pp. 97-107.

Proverbio, D. et al. (2022). "Model-based assessment of COVID-19
epidemic dynamics by wastewater analysis". In: _Science of the Total
Environment_ 827, p. 154235.

Riley, S. et al. (2020). "REal-time Assessment of Community
Transmission (REACT) of SARS-CoV-2 virus: study protocol". In:
_Wellcome Open Research_ 5.

Shah, S. et al. (2022). "Wastewater surveillance to infer COVID-19
transmission: A systematic review". In: _Science of The Total
Environment_ 804, p. 150060.

Tlhagale, M. et al. (2022). "Establishment of local wastewater-based
surveillance programmes in response to the spread and infection of
COVID-19-case studies from South Africa, the Netherlands, Turkey and
England". In: _Journal of Water and Health_ 20.2, pp. 287-299.
---

# Acknowledgments

.pull-left[
&lt;center&gt;&lt;img src=./img/Paper1.png width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

.pull-right[
**Guangquan Li** (Northumbria University)

**Radka Jersakova**, **Callum Mole**, **Camila Rangel Smith**  (Alan Turing Institute)

**Barry Rowlingson**, **Peter Diggle** (Lancaster University)

**Sylvia Richardson** (MRC Biostatistics Unit, Cambridge)

**George Nicholson**, **Chris Holmes** (Oxford University)

**Fatemeh Torabi** (Swansea University)

**Matt Wade**, **Hubert Denise**, **William Rowe**, **Daniel James**, **Jasmine Grimsley** (UKHSA)

.pull-left[
&lt;center&gt;&lt;img src=./img/qrcode_atlas.png width='35%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
.center[Scan to access the atlas]
]


.pull-right[

&lt;center&gt;&lt;img src=./img/qrcode_github.png width='35%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
.center[Scan to access the code and data]
]
]

    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script src="https://platform.twitter.com/widgets.js"></script>
<script>var slideshow = remark.create({
"navigation": {
"scroll": false
},
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
// add `data-at-shortcutkeys` attribute to <body> to resolve conflicts with JAWS
// screen reader (see PR #262)
(function(d) {
  let res = {};
  d.querySelectorAll('.remark-help-content table tr').forEach(tr => {
    const t = tr.querySelector('td:nth-child(2)').innerText;
    tr.querySelectorAll('td:first-child .key').forEach(key => {
      const k = key.innerText;
      if (/^[a-z]$/.test(k)) res[k] = t;  // must be a single letter (key)
    });
  });
  d.body.setAttribute('data-at-shortcutkeys', JSON.stringify(res));
})(document);
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>
<style>
.logo {
  background-image: url("assets/beamer/Logo.png");
  background-size: 15% 10%;
  background-repeat: no-repeat;
  position: absolute;
  top:  0.5%; /* 1.135em */
  left: 80%;
  width: 130%;
  height: 60%;
  z-index: 0;
}
</style>

<script>
document
  .querySelectorAll(
    '.remark-slide-content' +
    ':not(.title-slide)' +
    // add additional classes to exclude here, e.g.
    // ':not(.inverse)' +
    ':not(.hide-logo)' +
    ':not(.thankyou-michelle)' +
    ':not(.thankyou-barney)'
  )
  .forEach(el => {
    el.innerHTML += '<div class="logo"></div>';
  });
</script>


<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
