<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Spatio-temporal Bayesian models for environmental epidemiology</title>
    <meta charset="utf-8" />
    <meta name="author" content="Marta Blangiardo" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/panelset/panelset.css" rel="stylesheet" />
    <script src="libs/panelset/panelset.js"></script>
    <script src="libs/kePrint/kePrint.js"></script>
    <link href="libs/lightable/lightable.css" rel="stylesheet" />
    <!-- (Re)Defines a bunch of LaTeX commands that can then be used directly in the .Rmd file as '\command{...}' -->
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      TeX: {
        /* This enables color macros */
        extensions: ["color.js"],
        Macros: {
          /* Probability & mathematical symbols */
          Pr: "{\\style{font-family:inherit; font-size: 110%;}{\\text{Pr}}}",
          exp: "{\\style{font-family:inherit; font-size: 105%;}{\\text{exp}}}",
          log: "{\\style{font-family:inherit; font-size: 105%;}{\\text{log}}}",
          ln: "{\\style{font-family:inherit; font-size: 105%;}{\\text{ln}}}",
          logit: "{\\style{font-family:inherit; font-size: 100%;}{\\text{logit}}}",
          HR: "{\\style{font-family:inherit; font-size: 105%;}{\\text{HR}}}",
          OR: "{\\style{font-family:inherit; font-size: 105%;}{\\text{OR}}}",
          E: "{\\style{font-family:inherit; font-size: 105%;}{\\text{E}}}",
          Var: "{\\style{font-family:inherit; font-size: 105%;}{\\text{Var}}}",
          Cov: "{\\style{font-family:inherit; font-size: 105%;}{\\text{Cov}}}",
          Corr: "{\\style{font-family:inherit; font-size: 105%;}{\\text{Corr}}}",
          DIC: "{\\style{font-family:inherit; font-size: 105%;}{\\text{DIC}}}",
          se: "{\\style{font-family:inherit; font-size: 100%;}{\\text{se}}}",
          sd: "{\\style{font-family:inherit; font-size: 100%;}{\\text{sd}}}",
          kld: "{\\style{font-family:inherit; font-size: 100%;}{\\text{kld}}}",
          /* Distributions */
          dnorm: "{\\style{font-family:inherit;}{\\text{Normal}}}",
          dt: "{\\style{font-family:inherit;}{\\text{t}}}",
          ddirch: "{\\style{font-family:inherit;}{\\text{Dirichlet}}}",
          dmulti: "{\\style{font-family:inherit;}{\\text{Multinomial}}}",
          dbeta: "{\\style{font-family:inherit;}{\\text{Beta}}}",
          dgamma: "{\\style{font-family:inherit;}{\\text{Gamma}}}",
          dbern: "{\\style{font-family:inherit;}{\\text{Bernoulli}}}",
          dbin: "{\\style{font-family:inherit;}{\\text{Binomial}}}",
          dpois: "{\\style{font-family:inherit;}{\\text{Poisson}}}",
          dweib: "{\\style{font-family:inherit;}{\\text{Weibull}}}",
          dexp: "{\\style{font-family:inherit;}{\\text{Exponential}}}",
          dlnorm: "{\\style{font-family:inherit;}{\\text{logNormal}}}",
          dunif: "{\\style{font-family:inherit;}{\\text{Uniform}}}",
          /* LaTeX formatting */
          bm: ["{\\boldsymbol #1}",1],
          /* These create macros to typeset numbers in maths with the basic font */
          0: "{\\style{font-family:inherit; font-size: 105%;}{\\text{0}}}",
          1: "{\\style{font-family:inherit; font-size: 105%;}{\\text{1}}}",
          2: "{\\style{font-family:inherit; font-size: 105%;}{\\text{2}}}",
          3: "{\\style{font-family:inherit; font-size: 105%;}{\\text{3}}}",
          4: "{\\style{font-family:inherit; font-size: 105%;}{\\text{4}}}",
          5: "{\\style{font-family:inherit; font-size: 105%;}{\\text{5}}}",
          6: "{\\style{font-family:inherit; font-size: 105%;}{\\text{6}}}",
          7: "{\\style{font-family:inherit; font-size: 105%;}{\\text{7}}}",
          8: "{\\style{font-family:inherit; font-size: 105%;}{\\text{8}}}",
          9: "{\\style{font-family:inherit; font-size: 105%;}{\\text{9}}}",
          /* Health economics quantities */
          icer: "{\\style{font-family:inherit; font-size: 100%;}{\\text{ICER}}}",
          nb: "{\\style{font-family:inherit; font-size: 100%;}{\\text{NB}}}",
          ol: "{\\style{font-family:inherit; font-size: 100%;}{\\text{OL}}}",
          ceac: "{\\style{font-family:inherit; font-size: 100%;}{\\text{CEAC}}}",
          evpi: "{\\style{font-family:inherit; font-size: 100%;}{\\text{EVPI}}}",
          evppi: "{\\style{font-family:inherit; font-size: 100%;}{\\text{EVPPI}}}",
          evsi: "{\\style{font-family:inherit; font-size: 100%;}{\\text{EVSI}}}"
        }
      }
    });
    </script>
    <link rel="stylesheet" href="assets/beamer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">




class: title-slide

# Spatio-temporal Bayesian models for environmental epidemiology&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt; Methods and examples

## Marta Blangiardo

### [Environment and Health Statistics group](https://www.envstats.org/) | Imperial College London    

### EMS 2023, Warsaw University 

&lt;!-- Can also separate the various components of the extra argument 'params', eg as in 
### EMS 2023, Warsaw University, 7th July 2023, Spatio-temporal Bayesian models for environmental epidemiology
--&gt;

7th July 2023

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

&lt;center&gt;&lt;img src=./img/LogoMRC.png width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

---

layout: true  

.my-footer[ 
.alignleft[ 
&amp;nbsp; &amp;copy; Marta Blangiardo
]
.aligncenter[
Spatio-temporal Bayesian models for environmental epidemiology 
]
.alignright[
EMS 2023, NA 
]
] 


&lt;style&gt;
pre {
  overflow-x: auto;
}
pre code {
  word-wrap: normal;
  white-space: pre;
}
&lt;/style&gt;

---



# Environmental Epidemiology

- **Epidemiology**: The study of the distribution, causes and control of diseases in human population.
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
-  Disease risk depends on the classic epidemiological trio of person (in terms of genetics and behaviour), place and time.
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
- Environmental epidemiology focuses on linking .red[environmental hazards (exposures) to health outcomes]. 

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

--

Two main ingredients

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

  - Environmental exposure
  
  `\(\Rightarrow\)` a continuous field over the study area.

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

  - Health outcomes

  `\(\Rightarrow\)` cohort / administrative data.

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

Study designs typically used:
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
  1. Cohort studies - Focus on long-term effects (individual data)
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
  2. Small area studies - Focus on long-term effects (aggregated data)
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
  3. Time-series studies - Focus on short-term effects (aggregated data)

--

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

.center[.red[.large[Spatial and temporal dependencies are key.]]]

---

# Why accounting for spatial and temporal dependencies is important

.center[.content-box-blue[
**First law of geography** (W. Tobler, 1970): everything is related to everything else, but near things are more related than distant things
]
]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

When are we interested in the spatial and temporal components?

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Hypothesis generating perspective 

`\(\rightarrow\)` Are we explicitly interested in the spatial pattern of disease risk?

`\(\rightarrow\)` Do we want to evaluate temporal trends for each area?

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Is the spatial clustering and/or temporal trend a nuisance quantity that we wish to take into account but are not explicitly interested in?

`\(\rightarrow\)` Spatial / time series regression.

---
# Data/modelling/challenges in environmental epidemiology

&lt;center&gt;&lt;img src=./img/Areas.png width='60%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

--

In this talk
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
  - Wastewater modelling and COVID-19 prevalence (space-time geostatistical model; uncertainty propagation; data integration)
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
  - Temperature and COPD hospitalisations (space-time areal model; dose-response)
  
---

name: WBE

&lt;span style="display:block; margin-top: 250px ;"&gt;&lt;/span&gt;

.myblue[.center[.huge[
**Space-time modelling: wastewater based epidemiology**]]]

---

#Wastewater-based epidemiology

&lt;span style="display:block; margin-top: 50px ;"&gt;&lt;/span&gt;
&lt;center&gt;
.content-box-green[.center[.large[A collection of tools
and methods for surveillance and monitoring disease outbreaks using bio-
chemical analysis of wastewater samples as the primary outcome measure]]]
&lt;/center&gt;

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Wastewater has been used over the years as an epidemiological tool
  - no selection bias
  - cheap

- Wastewater based epidemiology started as a way to use wastewater to estimate exposure to chemicals in the population

  - Illicit drugs, psychoactive substances
  - Pesticides and other endocrine disruptors
  - Bacteria, viruses, fungi
  
Success stories:

  - retrospective prediction of outbreaks like hepatitis A and norovirus
  - polio - WHO guidelines for monitoring polio includes wastewater
  

---

# How does it work 

&lt;span style="display:block; margin-top: 50px ;"&gt;&lt;/span&gt;

&lt;center&gt;&lt;img src=./img/WW.png width='100%' title=''&gt;&lt;/center&gt;

---


# Wastewater during the COVID-19 pandemic

- WBE has been recognised as an economically efficient approach for disease surveillance 

- Methods to detect the presence of SARS-CoV-2 RNA in wastewater have
been developed in a number of countries 

- Scientific contributions to date have focused on the level of the sewage treatment works (STWs) where the
measurements of SARS-CoV-2 RNA from wastewater are obtained.

- No attempt has previously been made to develop a
spatially resolved model to estimate wastewater viral
concentration over a spatially continuous domain.

&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
--

## Aim

- Develop a statistical model to estimate viral concentration in wastewater at fine spatio-temporal resolutions covering the whole of England

---

# Modelling spatial data

- **Point-referenced data**: the exact location of the occurrences is known
  -  can be collected through specialized survey / monitoring network
  - if locations are **fixed** (monitoring stations, postcodes in an area) and the variable of interest is measured at each location (e.g. presence/absence of cases, pollution concentrations)
  
  `\(\Rightarrow\)` .red[geostatistical framework.]

--

- We want to .red[reconstruct a latent spatial (temporal) surface] `\(\class{red}{\mathbf{S}}\)` .red[from a finite set of noisy observations and their spatial (temporal) location.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Use the spatial dependence to **predict** values of the spatial
surface (together with associated uncertainty) at locations where
there are no observations.

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

.center[The common framework to geostatistical models is that of .red[**Gaussian
random fields**.]]

&lt;span style="display:block; margin-top: 70px ;"&gt;&lt;/span&gt;

---

# Gaussian random fields

- A spatial process is a Gaussian field if `\(\left(y(\bm s_1),\ldots, y(\bm s_n)\right)\)` follows a multivariate Normal distribution with mean `\(\bm \mu=\left(\mu(\bm s_{1}), \ldots, \mu(\bm s_{n})\right)\)` and spatially structured covariance matrix `\(\bm \Sigma\)`. 

--

- The spatial process is called .red[second-order stationary] if 

(i) `\(\bm \mu\)` is constant (i.e. `\(\mu(\bm s_{i}) = \mu\)` for each `\(i\)`) 

(ii) the spatial covariance function depends only on the distance vector `\((\bm s_{i}-\bm s_{j})\)`. 

--

- A stationary process is .red[isotropic] if the covariance does not depend on the direction but just on the Euclidean distance  `\(||\bm s_{i}-\bm s_{j}|| \in \mathbb{R}\)`.

--

- Several functions are available for the spatial covariance function (eg exponential,Mat&amp;eacute;rn, spherical, etc.) parameterized by some parameters (eg spatial variance, range, etc.).

---

# Data

- SARS-CoV-2 viral concentrations obtained from the 303 Sewage Treatment Works (STWs) for which the data were publicly available from the Environmental Monitoring for Health Protection (EMHP) wastewater surveillance programme.


.pull-left[
&lt;center&gt;&lt;img src=./img/PopCoverage.png width='100%' title='% pop coverage'&gt;&lt;/center&gt;
.center[Percentage population coverage]
]

.pull-right[
&lt;center&gt;&lt;img src=./img/STW.png width='61%' title='Location of STWs and their catchment areas'&gt;&lt;/center&gt;
.center[Location of STWs and their catchment areas]
]

---

# Data

- At each site three to four samples were collected per week between 1 June 2021 to 30 March 2022. 

--
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- We consider the weekly average of flow-normalised viral concentration measurements reported as the number of SARS-CoV-2 N1 gene copies per litre of wastewater (gc/L), obtained through reverse transcriptase quantitative polymerase chain reaction (RT-qPCR) analysis of the wastewater samples.

--
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- We also include socio-demographic, land use covariates and wastewater genomic data:

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

  1. Index of Multiple Deprivation (2019)
  2. Black, Asian and Minority Ethnic (BAME) proportion (Census, 2011)
  3. Land cover (Copernicus, 2018)
  4. Population density (ONS, 2019)
  5. Age structure (ONS, 2019)
  6. Percentage coverage of the SARS-CoV-2 genome in the sample
  7. Single-Nucleotide Polymorphism (SNP) data in the sample

---

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  u_i + v_t + z_{it}$$`

&lt;span style="display:block; margin-top: 60px ;"&gt;&lt;/span&gt;

.small[Li et al. A spatio-temporal framework for modelling wastewater concentration during the COVID-19 pandemic, Env Int 172 (2023)]
---

count:false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} ={\color{red}{\alpha_{region_{g_i}}}} + \bm{x}_i \bm{\beta} +  u_i + v_t + z_{it}$$`

- `\(\color{red}{\alpha_{region_{g_i}}}\)`: region specific intercept, modelled as exchangeable


&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

.small[Li et al. A spatio-temporal framework for modelling wastewater concentration during the COVID-19 pandemic, Env Int 172 (2023)]

---

count: false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} = \alpha_{region_{g_i}} + {\color{red}\bm{x}_i \bm{\beta}} +  u_i + v_t + z_{it}$$`

- `\(\color{red}{\bm{x}_i \bm{\beta}}\)`: vector of covariates and their effects


&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

.small[Li et al. A spatio-temporal framework for modelling wastewater concentration during the COVID-19 pandemic, Env Int 172 (2023)]

---

count: false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  {\color{red}u_i} + v_t + z_{it}$$`

- `\(\color{red}{u_i}\)`: spatial random effect with an exchangeable prior 


&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

.small[Li et al. A spatio-temporal framework for modelling wastewater concentration during the COVID-19 pandemic, Env Int 172 (2023)]

---


count: false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

### Model structure

The log-transformed number of gene copies per liter at catchment area `\(i=1, \ldots, 303\)` and week `\(t=1,\ldots, 44\)` is modelled as:

`$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$`

with `\(\sigma^2_y\)` as the measurement error variance. For the latent mean concentration `\(\mu_{it}\)` we specify a linear model:

`$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  u_i + {\color{red}v_t} + z_{it}$$`

- `\(\color{red}{v_t}\)`: temporal random effect modelled with first order random walk


&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

.small[Li et al. A spatio-temporal framework for modelling wastewater concentration during the COVID-19 pandemic, Env Int 172 (2023)]

---

# Spatio-temporal random effect

`$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  u_i + v_t + {\color{red}z_{it}}$$`

- `\(\color{red}{z_{it}}\)`: spatio-temporal random effect, modelled as a temporal autoregressive structure with spatially correlated innovations, similar to :
`\begin{align}
t = 1 \qquad \boldsymbol{z}_t &amp;\sim \text{Normal}(0,\Sigma_{z})\\
t &gt; 1 \qquad \boldsymbol{z}_t &amp;\sim \text{Normal}(\rho\boldsymbol{z}_{t-1},\Sigma_{z})
\end{align}`
with `\(\rho\)` as the temporal autoregressive coefficient. 

--

The  `\(\Sigma_{z}\)` matrix represents the spatial structure and is defined using a Mat&amp;eacute;rn covariance function.

`$$\text{Cov}(\bm{z}_i,\bm{z}_s)\ = \frac{\sigma^2_{z}}{\Gamma(\lambda)2^{\lambda-1}}(\kappa ||\bm{z}_i - \bm{z}_s||)^{\lambda}K_{\lambda}(\kappa ||\bm{z}_i - \bm{z}_s||)$$`

where `\(\Gamma(\lambda)\)` is the gamma function, `\(K_{\lambda}\)` is the modified Bessel function of second kind with order `\(\lambda\)`. The parameter `\(\kappa&gt;0\)` controls the range `\(r=\sqrt{8\lambda}/\kappa\)` and indicate the distance where the spatial correlation is close to 0.1. 

&lt;!-- k is scaling parameter, while lambda is the parameter of the Bessel function which governs the degree of smoothing (typically fixed)
--&gt;
---

# Spatial prediction

- We consider the population-weighted centroids of all the 32,844 Lower Super Output Areas (LSOAs) in England, these being the smallest areas 
for which all the covariates are available. 

Given observed data `\(\boldsymbol{y}\)` we sample `\(\boldsymbol{y}^\prime\)`, the viral concentrations at the LSOA centroids over 44 weeks, from the posterior predictive distribution:
`\begin{align}
\pi(\boldsymbol{y}^\prime \mid \boldsymbol{y}) &amp;= \int p(\boldsymbol{y}^\prime \mid \boldsymbol{\theta}) p(\boldsymbol{\theta} \mid \boldsymbol{y}) d\boldsymbol{\theta}\\
\boldsymbol{\theta} &amp;= (\alpha, \boldsymbol{\beta}, \boldsymbol{v}, \boldsymbol{g}, \boldsymbol{u}, \boldsymbol{z}, \sigma^2_y)\ .
\end{align}`


--

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;


The posterior predictive distribution at the LSOA centroids can be combined to return corresponding predictions at coarser geographical scales, as required. 

  - Here we consider all the Lower Tier Local Authorities in England and average the corresponding LSOA values, weighted by the population in each LSOA.


&lt;!--
- Implemented in R-INLA 
--&gt;

---


# Detection of areas at increased risk of outbreaks

We used the posterior predictive distribution to detect increases in the wastewater viral concentration. For each LTLA `\((l=1,\ldots,L)\)` an increase in viral concentration over two consecutive weeks is the event (E):
`\begin{eqnarray}
E = I(\mu_{lt} &gt; \mu_{l(t-1)} &gt; \mu_{l(t-2)})
\end{eqnarray}`
where `\(I(\cdot)\)` is the indicator function.

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
A high predictive probability, `\({\rm Prob}(E)&gt;c\)` for a pre-specified threshold `\(c\)`, would then trigger a warning of a potential outbreak. This approach has been used for public health surveillance in both high-income (Diggle et al, 2005) and low-to-middle-income (Diggle et al., 2007) countries. 

&lt;span style="display:block; margin-top: 250px ;"&gt;&lt;/span&gt;

.small[
Diggle et al. Point process methodology for on-line spatio-temporal disease surveillance, Environmetrics, 16 (2005): 423-434

Diggle et al. Spatial modelling and prediction of loa loa risk: decision making under uncertainty, Ann. Trop. Med. Parasitol., 101 (2007): 499-509

]
---

#Interactive dashboard
To visualise these rich model outputs, we have created a dynamic and interactive dashboard

&lt;span style="display:block; margin-top: -50px ;"&gt;&lt;/span&gt;

&lt;iframe src="https://b-rowlingson.gitlab.io/wwatlas/" width="140%" height="100%" data-external="1" style="transform: scale(.8);border:none"&gt;&lt;/iframe&gt;

---

# Wastewater viral concentration and COVID-19 prevalence

.pull-left[
- Several studies have evaluated the relationship between viral load in
wastewater and prevalence of COVID-19 disease (e.g. Shah et al. 2022, Proverbio et al. 2022)

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
- The relationship between viral concentration in
wastewater and COVID-19 prevalence appears to be **complex
and varying over both space and time.**

&lt;span style="display:block; margin-top: 200px ;"&gt;&lt;/span&gt;
.small[
Shah et al. Wastewater surveillance to infer COVID-19 transmission: A systematic review. Science of The Total Environment 804 (2022): 150060;

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

Proverbio et al. Model-based assessment of COVID-19 epidemic dynamics by wastewater analysis. Science of the Total Environment 827 (2022): 154235.]
]



.pull-right[&lt;center&gt;&lt;img src=./img/fig6.jpg width='50%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;]

---

# Wastewater in the post-pandemic era to monitor COVID

- Data on randomised surveys and testing become more sparse
&lt;span style="display:block; margin-top: -5px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` Provide information on coarse geographical resolution (e.g. regional or national)

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Wastewater can track prevalence well and is cheaper to measure
&lt;span style="display:block; margin-top: -5px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` A monitoring system needs to be maintained 


--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

What data do we have (in the context of England)?

&lt;!--a. Assuming we have wastewater viral concentrations regularly measured at STWs across the country

`\(\rightarrow\)` we can use the geostatistical model to predict the LTLA weekly concentration.

b. .red[LTLA weekly prevalence] estimates are only available up to a certain time point, but estimates at the weekly-coarse level remain available --&gt;


&lt;center&gt;&lt;img src=./img/WWCovid.png width='60%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

.content-box-green[Can wastewater data help disaggregate coarse level prevalence to
a finer spatial scale?
]

---

# Modelling framework

- Coarse (national) and local (LTLA) models jointly fitted

- Wastewater predictions available from the geostatistical model

- Posterior distribution of COVID-19 prevalence obtained through Nicholson et al. (2022) 

--

### Coarse (national)
`\begin{align*}
\tilde{P}_{t} &amp;\sim \mbox{N}(\tilde{\mu}_{t},\tilde{\sigma}^2_{t})\\
\tilde{\mu}_{t} &amp;= \alpha + b_t + d_t \tilde{y}^{\prime}_{t}
\end{align*}`

where 

- `\(\tilde{P}_{t}\)` and `\(\tilde{\sigma}^2_{t}\)` are posterior mean and variance of the national aggregation of the prevalence (population weighted)
- `\(\tilde{y}^{\prime}_{t}\)` is a similar aggregation for wastewater concentration

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- `\(b_t\)` is modelled using a RW1
- `\(d_t\)` is modelled using a RW2

&lt;span style="display:block; margin-top: 80px ;"&gt;&lt;/span&gt;

.small[Nicholson et al. Local prevalence of transmissible SARS-CoV-2 infection: an integrative causal model for debiasing fine-scale targeted testing data, Nature Microbiology 7 (2022): 97–107.]

---

# Modelling framework

- Coarse (national) and local (LTLA) models jointly fitted

- Wastewater predictions available from the geostatistical model

- Posterior distribution of COVID-19 prevalence oobtained through Nicholson et al. (2022) 

--

### Local
`\begin{align*}
p_{lt} &amp;\sim \mbox{N}(\mu_{lt},\sigma^2_{lt})\\
\mu_{lt} &amp;= (\alpha + u_l) + (b_t + v_{lt}) + (d_t + m_l)y^{\prime}_{lt}\\
\bm{y}^{\prime}_{t} &amp;\sim \mbox{MVN}(\bm{g}_t, \bm{S}_t)
\end{align*}`

where 

- `\(p_{lt}\)` and `\(\sigma^2_{lt}\)` are posterior mean and variance of the COVID-19 prevalence 
- `\(\bm{g}_t\)` and `\(\bm{S}_t\)` are the posterior mean and covariance matrix from the wastewater model
- `\(u_{l}\)`, `\(v_{lt}, m_l\)` are modelled as exchangeable

&lt;span style="display:block; margin-top: 80px ;"&gt;&lt;/span&gt;

.small[Nicholson et al. Local prevalence of transmissible SARS-CoV-2 infection: an integrative causal model for debiasing fine-scale targeted testing data, Nature Microbiology 7 (2022): 97–107.]

---

# Implementation

- Both the local and national components are jointly fitted

- Alternative models: 

1\. No wastewater

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` temporal trend for the coarse level model `\(\tilde{\mu}_{t} = \alpha + b_t + d_t\)`
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` spatial and temporal trend for the local level model `\(\mu_{lt} = (\alpha + u_l) + (b_t + v_{lt})\)`

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;

2\. No temporal varying coefficients
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` linear relationship with wastewater `\(\tilde{\mu}_{t} = \alpha + b_t + d \tilde{y}^{\prime}_{t}\)`
&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;
`\(\rightarrow\)` spatial-varying coefficient for wastewater `\(\mu_{lt} = (\alpha + u_l) + (b_t + v_{lt}) + (d + m_l)y^{\prime}_{lt}\)`

--

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;

- *Study period*: 40 weeks

- **Training window**: first 20 weeks (.red[LTLA + national weekly prevalence] + .blue[LTLA + national weekly wastewater])

- **Disaggregation window**: subsequent 5/10/15/20 weeks (.red[national weekly prevalence] + .blue[LTLA + national weekly wastewater])

---

# Results

.pull-left[&lt;center&gt;&lt;img src=./img/Results.png width='200%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

.pull-right[
- Using wastewater data helps to improve the quality of the disaggregated LTLA prevalence (lower RMSE and better coverage) consistently across all disaggregation horizons.

- Not large difference between the model 2 and model 3

- Some geographical variation

]

&lt;span style="display:block; margin-top: -20px ;"&gt;&lt;/span&gt;
&lt;center&gt;&lt;img src=./img/Boxplot.png width='80%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;


---

# Results

.pull-left[&lt;span style="display:block; margin-top: 40px ;"&gt;&lt;/span&gt;
&lt;center&gt;&lt;img src=./img/Heatmap.png width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]


.pull-right[&lt;span style="display:block; margin-top: 5px ;"&gt;&lt;/span&gt;
&lt;center&gt;&lt;img src=./img/Scatterplot.png width='80%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

`\(\rightarrow\)` Some periods (e.g. Christmas are showing larger differences)

`\(\rightarrow\)` Some areas are consistently underestimated/overestimated

---

# Discussion

- Relationship with disease is not trivial. In a descriptive way we found a spatial and temporal dynamicity and evidence of non-linearity. It could depend on several factors like vaccination, variants and metereology.

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- The proposed framework for disaggregation demonstrates the added-value of wastewater data to inform local prevalence

`\(\rightarrow\)` Continuous running of randomised surveys at a reduced capacity to provide coarse prevalence estimates remains a key pillar in COVID monitoring, anchoring the disaggregation and the relationship

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- The wastewater-prevalence relationship seems stable in time over the study period but will likely need to be recalibrated over longer times

`\(\rightarrow\)` occasional running of randomised surveys at a fine spatial scale is required

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

`\(\rightarrow\)` impact on the results will need to be evaluated

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Starting point but a long way to go

`\(\rightarrow\)` should be extended to include other data sources, such as syndromic and secondary care data, to enable quantitatively accurate monitoring of disease evolution across a population.

---

name: COPD

&lt;span style="display:block; margin-top: 250px ;"&gt;&lt;/span&gt;

.myblue[.center[.huge[
**Space-time modelling: Temperature and Health**]]]



---

# COPD and temperature
.panelset[
.panel[.panel-name[COPD]

&lt;center&gt;&lt;img src=./img/COPD_lung.png width='60%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

- In England, 115,000 emergency admissions and 24,000 deaths per year.
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
- COPD exacerbations: Bacteria, viruses and air-pollution.
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
- The role of temperature is unclear. 

]

.panel[.panel-name[Temperature and Health]

.pull-left[
- Typically U-shaped relationship between temperature and mortality.
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
- Cold, dry air or hot air can trigger a flare-up.
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
- Different confounding, different lags across different temperatures.
]

.pull-right[

&lt;center&gt;&lt;img src=./img/TemperatureHealth.png width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

]
]

---

# Aim

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;
.center[.content-box-green[Examine the effect of warm temperatures on COPD hospitalizations.]]

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;

Secondary objectives:
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
- Examine effect modification by age, sex and space.
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
- Assess how contextual characteristics, such as green space, average temperature, deprivation and urbanicity affect population vulnerability. 
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;
- Calculate the number of COPD cases attributable to temperature.

---

# Outcome and exposure

.pull-left[
### Outcome
- COPD hospitalization (ICD10 J40-44) 2007-2018 (NHS digital).
- Individual data/ 100m grid spatial resolution.
- Summer months.

### Exposure
- Daily maximum temperature 2007-2018 at 1km grid from MetOffice.
- lag0-2.
]

.pull-right[
&lt;center&gt;&lt;img src=./img/temperatureLTLA.png width='80%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

---

# Case-crossover design

- Epidemiological study design for transient environmental exposures
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;    
- Sample from same month and days
&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;    
- Exposure period 0-2 days before the hospital episode (mean temperature)

&lt;center&gt;&lt;img src=./img/FigCaseCrossOver.png width='70%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

- Adding a spatial dimension to account for modifiers
---

# Confouders and spatial effect modifiers

.panelset[
.panel[.panel-name[Confounders]

&lt;table class=" lightable-material lightable-striped lightable-hover" style='font-family: "Source Sans Pro", helvetica, sans-serif; margin-left: auto; margin-right: auto;'&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style="text-align:left;"&gt; Confounder &lt;/th&gt;
   &lt;th style="text-align:left;"&gt; Source &lt;/th&gt;
   &lt;th style="text-align:left;"&gt; Space &lt;/th&gt;
   &lt;th style="text-align:left;"&gt; Time &lt;/th&gt;
   &lt;th style="text-align:left;"&gt; Year &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; PM2.5 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; MetOffice &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 1Km &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; daily &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 2007-2018 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; O3 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; MetOffice &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 1Km &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; daily &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 2007-2018 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Relative Humidity &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; MetOffice &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 10Km &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; daily &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 2007-2018 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Holidays &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; ONS &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; nationwide &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; daily &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 2007-2018 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;
]

.panel[.panel-name[Effect modifiers]

&lt;center&gt;&lt;img src=./img/SpatEffMod.png width='70%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]
]

---

# Step 1: Find the linear threshold

Let `\(Y_{tj}\)` be the case-control identifier for the COPD hospitalisation for the event (case or control) at time `\(t\)`, in the `\(j-th\)` case-control group:
  
`\begin{align}
Y_{tj} &amp;\sim \mbox{Poisson}(\mu_{tj})\\
\log(\mu_{tj}) &amp;= \alpha_1 I(X_{1t} &lt; c) X_{1t} + \alpha_2 I(X_{1t} \geq c) X_{1t} + \sum_{m=1}^4\beta_mZ_{mt} + u_{j}\\
u_j &amp;\sim N(0, 100)\\
\alpha_1, \alpha_2, \beta_1, \dots \beta_4 &amp; \sim N(0, 1)\\
\end{align}`

`\(c\)` is the temperature threshold,  `\(50-\)`th, `\(55-\)`th, ...,  `\(90-\)`th,  `\(95-\)`th percentile. 

--

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

We ran the above model for the different temperature thresholds and chose the one which minimises the WAIC.

---

# Step 2a: Effect modification by age and sex
  
Similarly we fitted the previous model for the different sex-age groups (male/female and `\(&lt;65\)`, `\(65-85\)`, `\(&gt;85\)` ). Consider `\(g\)` the index for the age-sex groups:

`\begin{align*}
Y_{tjg} &amp;\sim \text{Poisson}(\mu_{tjg})\\
\log(\mu_{tjg}) &amp;= \alpha_1 I(X_{1tg} &lt; c^{\star}) X_{1tg} + \alpha_2 I(X_{1tg}\geq c^{\star})X_{1tg} + \sum_{m=1}^4\beta_mZ_{mtg} + u_{j}\\
u_j &amp;\sim N(0, 100)\\
\alpha_1, \alpha_2, \beta_1, \dots \beta_4 &amp; \sim N(0, 1)\\
\end{align*}`

---

# Step 2b. Spatial Effect modification
.pull-left[    
`\begin{align*}
Y_{tj} &amp; \sim \text{Poisson}(\mu_{tj})\\
\log(\mu_{tj}) &amp; = \alpha_1 I(X_{1t}&lt; c^{\star})X_{1t} + \alpha_{2s_j} I(X_{1t}\geq c^{\star})X_{1t} +\\ 
&amp; \sum_{m=1}^4\beta_mZ_{mt} + u_{j}\\
\alpha_{2s} &amp;= \alpha_2 + \sum_{m=1}^8 \gamma_q H_{sq} + v_s + b_s\\
v_s &amp;\sim N(0, \sigma_2)\\
b_s|b_{-s} &amp;\sim N\left(\frac{\sum_{s \sim r} w_{rs} b_s}{\sum_{s \sim r} w_{rs}},        \frac{\sigma^2}{\sum_{s \sim r} w_{rs}}\right)\\
\end{align*}`

where `\(\bm{H}_s\)` is a vector of the following covariates at the Lower Tier Local Authority level (LTLA):

1\. Quintiles of deprivation
2\. Proportion of green space
3\. Urbanicity index
4\. Average temperature
]

.pull-right[
  &lt;center&gt;&lt;img src=./img/London.jpg width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;
.small[
Konstantinoudis et al., Asthma hospitalisations and heat exposure in England: a case–crossover study during 2002–2019, Thorax (2023), in press.
]

---

# Results

.panelset[.panel[.panel-name[Threshold selection and effect modification]

- In the adjusted model, the 80th percentile of the temperature (23.2C) was the threshold minimising the WAIC

- The increase in the COPD hospitalisation risk for every 1°C increase in temperatures above 23.2C was 0.47% (95% CrI 0.16% to 0.75%) 


.pull-left[

&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;
Effect modification by age and sex: 

- unadjusted models show increased risk which substantially reduces when the confounders are included
&lt;span style="display:block; margin-top: 15px ;"&gt;&lt;/span&gt;
- Evidence of an increased risk for females above 65 years old
]

.pull-right[
&lt;center&gt;&lt;img src=./img/ConfoundersRR.jpg width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;
]
]

.panel[.panel-name[Spatial effect modifiers]

.pull-left[
&lt;center&gt;&lt;img src=./img/SVC.png width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

&lt;center&gt;&lt;img src=./img/SVC_adj.png width='100%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

]

.pull-right[

- Evidence of spatial variation
&lt;span style="display:block; margin-top: 30px ;"&gt;&lt;/span&gt;
- Not enough evidence of effects of the covariates in explaining the spatial variation

&lt;table class=" lightable-material lightable-striped lightable-hover" style='font-family: "Source Sans Pro", helvetica, sans-serif; margin-left: auto; margin-right: auto;'&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style="text-align:left;"&gt; Modifier &lt;/th&gt;
   &lt;th style="text-align:left;"&gt; PercentChange &lt;/th&gt;
   &lt;th style="text-align:right;"&gt; Probability &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Urban &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; −1.57 (−4.16 to 0.96) &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 0.12 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; Avg Temp &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; −0.41 (−1.49 to 0.71) &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 0.22 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;
]
]
]

---

# Population Attributable Fraction (PAF)

- Typically used in epidemiological studies to evaluate the number of health end points which happened due to the exposure/risk factor.

.pull-left[
- Generally estimated over the entire study region

- Here we used the model with spatially-varying coefficients to estimate a local PAF using the following three steps

1. Obtain the cumulative exposure-outcome relative risk `\(\text{RR}_s = \exp\big(\alpha_{2s}(X_{1, s, \text{max}} - c^*)\big)\)`
2. Calculate `\(\text{AF}_s = (\text{RR}_s - 1)/\text{RR}_s\)`

3. Then `\(\text{PAF}_s = \text{AF}_s (n_s/N_s)\)`

where `\(N_s\)` and `\(n_s\)` are the total number of hospitalisations and those at days above 23.2C 
]

.pull-right[

&lt;center&gt;&lt;img src=./img/PAF.jpg width='70%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

- Overall 7.8% (95% CrI 6.7% to 8.8%) of the total COPD hospitalisations in summer (2007-2019) are due to temperature above 23.2C
- Clear spatial structure (more than 8% in East Midlands, East of England, London and South East, while it is below 5% in the South West).

]

---

# Discussion

- Spatial case-crossover model allows to take advantage of individual level data when the individual level confounders are not available (as long as they do not change quickly in time)

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Extended to investigate spatially effect modifiers (could be done for temporal effect modifiers too)

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Computationally intensive

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Evidence that COPD hospitalisations increase with warmer temperatures and as temperatures consistently increase, public health systems should be alerted and prepared to challenge the increased COPD hospitalisation burden. 

---
# General remarks

- Spatial and temporal dependencies are crucial to account for in environmental epidemiology

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

`\(\rightarrow\)` they add a level of understanding on the phenomenon under study and help reduce biases

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

`\(\rightarrow\)` using the spatio-temporal structure (and potentially covariates) the model can predict at any location in the study region

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Increased flexibility, but also additional statistical challenges

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

`\(\rightarrow\)` parameter structure

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

`\(\rightarrow\)` computationally intensive

&lt;span style="display:block; margin-top: 10px ;"&gt;&lt;/span&gt;

`\(\rightarrow\)` transferability

&lt;span style="display:block; margin-top: 20px ;"&gt;&lt;/span&gt;

- Reporting is challenging, particularly when dealing with stakeholders

---

# Acknowledgments

.pull-left[
.red[Temperature and COPD:]

&lt;center&gt;&lt;img src=./img/MRClogo.png width='60%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

**Garyfallos Konstantinoudis**, **Cosetta Minelli** (ICL)

**Antonio Gasparrini** (LSHTM)

**Ana Maria Vicedo Cabrera** (University of Bern)

**Joan Ballester** (ISGlobal)

]

.pull-right[
.red[Wastewater based epidemiology:]

&lt;center&gt;&lt;img src=./img/Turing.png width='70%' title='INCLUDE TEXT HERE'&gt;&lt;/center&gt;

**Guangquan Li** (Northumbria University)

**Radka Jersakova**, **Callum Mole**, **Camila Rangel Smith**  (Alan Turing Institute)

**Barry Rowlingson**, **Peter Diggle** (Lancaster University)

**Sylvia Richardson** (MRC Biostatistics Unit, Cambridge)

**George Nicholson**, **Chris Holmes** (Oxford University)

**Fatemeh Torabi** (Swansea University)

**Matt Wade**, **Hubert Denise**, **William Rowe**, **Daniel James**, **Jasmine Grimsley** (UKHSA)

]
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script src="https://platform.twitter.com/widgets.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
// add `data-at-shortcutkeys` attribute to <body> to resolve conflicts with JAWS
// screen reader (see PR #262)
(function(d) {
  let res = {};
  d.querySelectorAll('.remark-help-content table tr').forEach(tr => {
    const t = tr.querySelector('td:nth-child(2)').innerText;
    tr.querySelectorAll('td:first-child .key').forEach(key => {
      const k = key.innerText;
      if (/^[a-z]$/.test(k)) res[k] = t;  // must be a single letter (key)
    });
  });
  d.body.setAttribute('data-at-shortcutkeys', JSON.stringify(res));
})(document);
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
