---
title: "Spatio-temporal Bayesian models for environmental epidemiology"
subtitle: "Methods and examples"
author: Marta Blangiardo
institute: "[Environment and Health Statistics group](https://www.envstats.org/) | Imperial College London"
params: 
   conference: "EMS 2023"
   location: "Warsaw University"
   date: 7 July 2023
   short_title: "Spatio-temporal Bayesian models for environmental epidemiology"
output:    
  xaringan::moon_reader: 
    includes: 
       in_header: "assets/latex_macros.html" 
       # This line adds a logo based on the format selected in the file 'assets/include_logo.html'
       # NB: the actual options (eg placement of the logo and actual logo file) can be changed there
      # after_body: "assets/insert-logo.html"
    seal: false
    yolo: no
    lib_dir: libs
    nature:
      beforeInit: ["https://platform.twitter.com/widgets.js"]
      highlightStyle: github
      highlightLines: yes
      countIncrementalSlides: no
      ratio: '16:9'
      titleSlideClass:
      - center
      - middle
    self_contained: false 
    css:
    - "assets/beamer.css"

---

```{r global_options, echo = FALSE, include = FALSE}
options(width = 999)
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE,
                      cache = FALSE, tidy = FALSE, size = "small")
```
```{r echo=F,message=FALSE,warning=FALSE,comment=NA}
# Sources the R file with all the relevant setup and commands
source("assets/setup.R")
library(INLA)

# Stuff from 'xaringanExtra' (https://pkg.garrickadenbuie.com/xaringanExtra)
# This allows the use of panels (from 'xaringanExtra')
xaringanExtra::use_panelset()
# This allows to copy code from the slides directly
#xaringanExtra::use_clipboard()
# This freezes the frame for when there's a gif included
#xaringanExtra::use_freezeframe()

# Defines the path to the file with the .bib entries (in case there are references)
bibfile=RefManageR::ReadBib("~/Dropbox/Conferences-Talks/EMS2023/refs.bib",check = FALSE)
```
class: title-slide

# `r rmarkdown::metadata$title``r vspace("10px")` `r rmarkdown::metadata$subtitle`

## `r rmarkdown::metadata$author`

### `r rmarkdown::metadata$institute`    

### `r rmarkdown::metadata$params$conference`, `r rmarkdown::metadata$params$location` 

<!-- Can also separate the various components of the extra argument 'params', eg as in 
### `r paste(rmarkdown::metadata$params, collapse=", ")`
-->

`r ifelse(is.null(rmarkdown::metadata$params$date),format(Sys.Date(),"%e %B %Y"),rmarkdown::metadata$params$date)`

`r vspace("20px")`

`r include_fig("LogoMRC.png")`

---

layout: true  

.my-footer[ 
.alignleft[ 
&nbsp; &copy; Marta Blangiardo
]
.aligncenter[
`r rmarkdown::metadata$params$short_title` 
]
.alignright[
`r rmarkdown::metadata$params$conference`, `r short_date` 
]
] 

```{css,echo=FALSE, eval=FALSE}
.red {
  color: red;
}
.blue {
  color: 0.14 0.34 0.55;
}

.content-box-blue { background-color: #F0F8FF; }

}
```
<style>
pre {
  overflow-x: auto;
}
pre code {
  word-wrap: normal;
  white-space: pre;
}
</style>


---

# Environmental Epidemiology

- **Epidemiology**: The study of the distribution, causes and control of diseases in human population.
`r vspace("20px")`
-  Disease risk depends on the classic epidemiological trio of person (in terms of genetics and behaviour), place and time.
`r vspace("20px")`
- Environmental epidemiology focuses on linking .red[environmental hazards (exposures) to health outcomes]. 

`r vspace("20px")`

--

Two main ingredients

`r vspace("20px")`

  - Environmental exposure
  
  $\Rightarrow$ a continuous field over the study area.

`r vspace("20px")`

  - Health outcomes

  $\Rightarrow$ cohort / administrative data.

--

`r vspace("20px")`

- Study designs typically used:
`r vspace("20px")`
  1. Cohort studies - Focus on long-term effects (individual data)
`r vspace("20px")`
  2. Small area studies - Focus on long-term effects (aggregated data)
`r vspace("20px")`
  3. Time-series studies - Focus on short-term effects (aggregated data)

--

`r vspace("20px")`

.center[.red[.large[Spatial and temporal dependencies are key.]]]

---

# Why accounting for spatial and temporal dependencies is important

.center[.content-box-blue[
**First law of geography** (W. Tobler, 1970): everything is related to everything else, but near things are more related than distant things
]
]

`r vspace("20px")`

When are we interested in the spatial and temporal components?

`r vspace("20px")`

- Hypothesis generating perspective 

$\rightarrow$ Are we explicitly interested in the spatial pattern of disease risk?

$\rightarrow$ Do we want to evaluate temporal trends for each area?

--

`r vspace("20px")`

- Is the spatial clustering and/or temporal trend a nuisance quantity that we wish to take into account but are not explicitly interested in?
$\rightarrow$ Spatial / time series regression.

---
# Data/modelling/challenges in environmental epidemiology

`r include_fig("Areas.png",width="60%")`

`r vspace("20px")`

In this talk

  - Source apportionment of air pollution and health (times series + risk assessment)
`r vspace("20px")`
  - Wastewater modelling (space-time modelling)
`r vspace("20px")`
  - Temperature and COPD (non linear relationship + risk assessment)

---

name: SA

`r vspace("250px")`

.myblue[.center[.huge[
**Time series analysis: source apportionment**]]]


---

# Air pollution and health

- Outdoor air pollution is carcinogenic to humans (IARC 2013);
`r vspace("10px")`
-  Air pollution causes over 6.5 million deaths each year globally, and this number is increasing (Fuller et al. 2022).
`r vspace("10px")`
- Outdoor  PM causes a reduction in life expectancy of average population by approximately a year in Europe (WHO 2006);
`r vspace("10px")`
- Fossil fuel air pollution is responsible for 1 in 5 deaths worldwide (Vohra et al. 2021).
`r vspace("10px")`
- Synergistic effects between heat and air pollution exposure for all-causes as well as cardio-respiratory mortality/hospital admissions (Anenberg et al. 2020).
`r vspace("10px")`
- The triad of pollution, climate change, and biodiversity loss are the key global environmental issues of our time (Fuller et al. 2022). 

`r vspace("20px")`

.center[.content-box-blue[
Particulate matter - PM and gasses such as nitrogen oxide/dioxide and ozone are among the most studied pollutants in environmental epidemiology  
]
]

---

# Source apportionment

- Particulate matter (PM) is made up of different components coming
from several sources, which might have different level of toxicity 

- Identifying these sources, known as **Source Apportionment** (SA), is a key element in order to implement effective policies to improve air quality and population health.

`r include_fig("Mixture.jpg", width="50%")`

<center>
.content-box-green[.center[.large[*Aim*: to apportion particle number size distributions (PNSD) and
investigate the short-term health effects of the identified sources]]]
</center>

---

# Data

- Particle size distributions, collected at the background monitor station of North Kensington in central London. The particle size range is between 16.5-604.3 nm
`r vspace("20px")`
- Meteorological conditions (temperature, humidity)
`r vspace("20px")`
- Respiratory hospital admissions in children (0-16 years old), and
cardiovascular hospital admissions in elderly (+64 years old) in Greater London for the period 2014-2016

### Time series framework
- Typically used for investigating the short-term associations between exposures such as air pollution or weather variables, and health outcomes such as cardiovascular and respiratory morbidity and mortality.

- Data available at regular time intervals and with a natural order

`r include_fig("Airpoll-health.jpg", width="40%")`

.tiny[source: `r Citet(bibfile,"samoli2003investigating")`]

---

## What are the challenges?
.pull-left[
`r vspace("50px")`
- High dimensionality

`r vspace("20px")`

- Correlation 

`r vspace("20px")`

- Link with health
]

.pull-right[
`r include_fig("correlation-matrix-bin_1.jpg", width="120%")`
]

---

# Factor analysis

.pull-left[
Positive Matrix Factorisation commonly used for dimension reduction

$Y_{P \times T} = \Lambda_{P \times K}F_{K \times T}+ \epsilon_{P \times T}$
where 

- $\Lambda_{P \times K}$ are source profile densities;

- $F_{K \times T}$ are the contribution from the $k$ source to the total particle concentration.

Limitations:
- Number of sources defined a priori;
`r vspace("20px")`
- Uncertainty on the allocation not included;
`r vspace("20px")`
- Dependence in time/space not considered in the allocation.

]

.pull-right[

`r include_fig("SourceOutdoorPoll.jpg",width="80%")`
.center[.tiny[Source: .url[shan.org]]]
]


---
# Proposed approach 

- Bayesian non-parametric 
`r vspace("20px")`
- High degree of flexibility;
`r vspace("20px")`
  $\rightarrow$ number of sources estimated from the data;
`r vspace("10px")`
  $\rightarrow$ uncertainty fully propagated in the source allocation.

Following `r Cite(bibfile,"griffiths2011indian")`, `r Cite(bibfile, "ghahramani2005infinite")`
.pull-left[
\begin{align}
\textbf{X}_t &\sim N_D( (\textbf{Z}_t \circ \textbf{W}_t)\Phi,\sigma^2 I) \nonumber,\\
\textbf{W}_t &\sim N_K(0,I) \nonumber,\\
\phi_d &\sim N_K(\mu_\phi,\Sigma_\phi)\nonumber,\\
Z &\sim IBP(\alpha)\nonumber,\\
\alpha &\sim \text{Gamma}(\alpha_0,\beta_0),\nonumber\\
\sigma^2 &\sim \text{Gamma}^{-1}(b_0,b_1)
\end{align}
]

.pull-right[
DAG
]

---

# Characteristics of the model

- time is accounted for in the allocation 

- $\alpha$ governs the number of components

--

### Implementation

- The model is implemented in R-Nimble

- 2 chains, 100,000 iterations

### Sensitivity analysis

- Change the prior for $\alpha$

---

# Results: source profiles

.pull-left[
`r include_fig("Profiles.png")`
]


.pull-right[
`r include_fig("Heatmap.png")`
]

*Nucleation*: diesel peaks with associated traffic temporal
patterns

*Secondary*: Aged particles that have grown due to
the condensation of volatile gaseous compounds on
their surface

*Urban*: Aged traffic emissions, including also a mix
of other urban emissions such as biomass burning,
heating, cooking

*Traffic 1*: Particles associated with spark-ignition vehicles

*Traffic 2*: Particles associated with diesel vehicle emissions

---

# Link with health

We assessed the relative risk (RR) for hospitalization for respiratory diseases in children by including the SA posterior estimates into the health model


\begin{align*}
y_t &\sim \text{Poisson}(\mu_t E)\\
\log(\mu_{t})&= \beta_0 + \sum_k \beta_k f_{k(t-\ell)} + \sum_i s(v_{ti}, \psi_i) + \delta_{I_t} + \epsilon_t
\end{align*}

--

where $f_{k(t-\ell)}$ are the factors with specified lags

- Additionally, temporal long term components are included through $\epsilon$

- Confounders are accounted for in a non-linear flexible way through splines
---

# Results: health effects

.pull-left[
`r include_fig("Children_Resp.jpg", width="50%")`
]

.pull-right[

- Clear differential impact of the different sources

$\rightarrow$ Nucleation and urban show evidence of a negative impact on health 

]
---

# Moving forward

- Understanding the impact of a two-stage vs joint model for sources and health

- Several lags might need to be considered

- Additional spatial component might be beneficial to understand the different impact of these sources

---

name: WBE

`r vspace("250px")`

.myblue[.center[.huge[
**Space-Time modelling: wastewater based epidemiology**]]]

---

#Wastewater-based epidemiology

`r vspace("50px")`
<center>
.content-box-green[.center[.large[A collection of tools
and methods for surveillance and monitoring disease outbreaks using bio-
chemical analysis of wastewater samples as the primary outcome measure]]]
</center>

--

`r vspace("20px")`

- Wastewater has been used over the years as an epidemiological tool
  - no selection bias
  - cheap

- Wastewater based epidemiology started as a way to use wastewater to estimate exposure to chemicals in the population

  - Illicit drugs, psychoactive substances
  - Pesticides and other endocrine disruptors
  - Bacteria, viruses, fungi
  
Success stories:

  - retrospective prediction of outbreaks like hepatitis A and norovirus
  - polio - WHO guidelines for monitoring polio includes wastewater
  

---

# How does it work 

`r vspace("50px")`

`r include_fig("WW.png",width="100%", title="")`

---


# Wastewater during the COVID-19 pandemic

- WBE has been recognised as an economically efficient approach for disease surveillance `r Cite(bibfile,"manuel2022strengthening")`

- Methods to detect the presence of SARS-CoV-2 RNA in wastewater have
been developed in a number of countries `r Cite(bibfile,"tlhagale2022establishment")`

- Scientific contributions to date have focused on the level of the sewage treatment works (STWs) where the
measurements of SARS-CoV-2 RNA from wastewater are obtained.

- No attempt has previously been made to develop a
spatially resolved model to estimate wastewater viral
concentration over a spatially continuous domain.

`r vspace("40px")`
--

## Aim

- Develop a statistical model to estimate viral concentration in wastewater at fine spatio-temporal resolutions covering the whole of England

---

# Modelling spatial data

- **Point-referenced data**: the exact location of the occurrences is known
  -  can be collected through specialized survey / monitoring network...
  - if locations are **fixed** (monitoring stations, postcodes in an area) and the variable of interest is measured at each location (e.g. presence/absence of cases, pollution concentrations)
  
  $\Rightarrow$ .red[geostatistical framework.]

--

.center[.content-box-blue[
**First law of geography** (W. Tobler, 1970): everything is related to everything else, but near things are more related than distant things `r Citet(bibfile,"miller2004tobler")`
]
]

--

- We want to .red[reconstruct a latent spatial (temporal) surface] $\class{red}{\mathbf{S}}$ .red[from a finite set of noisy observations and their spatial (temporal) location.]

`r vspace("20px")`

- Use the spatial dependence to **predict** values of the spatial
surface (together with associated uncertainty) at locations where
there are no observations.

--

`r vspace("20px")`

.center[The common framework to geostatistical models is that of .red[**Gaussian
random fields**.]]


---

# Data

- SARS-CoV-2 viral concentrations obtained from the 303 Sewage Treatment Works (STWs) for which the data were publicly available from the Environmental Monitoring for Health Protection (EMHP) wastewater surveillance programme.


.pull-left[
`r include_fig("PopCoverage.png", width="100%", title="% pop coverage")`
.center[Percentage population coverage]
]

.pull-right[
`r include_fig("STW.png", width="61%", title="Location of STWs and their catchment areas")`
.center[Location of STWs and their catchment areas]
]

---

# Data

- At each site three to four samples were collected per week between 1 June 2021 to 30 March 2022. 

--
`r vspace("20px")`

- We consider the weekly average of flow-normalised viral concentration measurements reported as the number of SARS-CoV-2 N1 gene copies per litre of wastewater (gc/L), obtained through reverse transcriptase quantitative polymerase chain reaction (RT-qPCR) analysis of the wastewater samples.

--
`r vspace("20px")`

- We also include socio-demographic, land use covariates and wastewater genomic data:

`r vspace("20px")`

  1. Index of Multiple Deprivation (2019)
  2. Black, Asian and Minority Ethnic (BAME) proportion (Census, 2011)
  3. Land cover (Copernicus, 2018)
  4. Population density (ONS, 2019)
  5. Age structure (ONS, 2019)
  6. Percentage coverage of the SARS-CoV-2 genome in the sample
  7. Single-Nucleotide Polymorphism (SNP) data in the sample

---

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

`r vspace("20px")`

### Model structure

The log-transformed number of gene copies per liter at catchment area $i=1, \ldots, 303$ and week $t=1,\ldots, 44$ is modelled as:

$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$

with $\sigma^2_y$ as the measurement error variance. For the latent mean concentration $\mu_{it}$ we specify a linear model:

$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  u_i + v_t + z_{it}$$
 
---

count:false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

`r vspace("20px")`

### Model structure

The log-transformed number of gene copies per liter at catchment area $i=1, \ldots, 303$ and week $t=1,\ldots, 44$ is modelled as:

$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$

with $\sigma^2_y$ as the measurement error variance. For the latent mean concentration $\mu_{it}$ we specify a linear model:

$$\mu_{it} ={\color{red}{\alpha_{region_{g_i}}}} + \bm{x}_i \bm{\beta} +  u_i + v_t + z_{it}$$

- $\color{red}{\alpha_{region_{g_i}}}$: region specific intercept, modelled as exchangeable

---

count: false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

`r vspace("20px")`

### Model structure

The log-transformed number of gene copies per liter at catchment area $i=1, \ldots, 303$ and week $t=1,\ldots, 44$ is modelled as:

$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$

with $\sigma^2_y$ as the measurement error variance. For the latent mean concentration $\mu_{it}$ we specify a linear model:

$$\mu_{it} = \alpha_{region_{g_i}} + {\color{red}\bm{x}_i \bm{\beta}} +  u_i + v_t + z_{it}$$

- $\color{red}{\bm{x}_i \bm{\beta}}$: vector of covariates and their effects

---

count: false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

`r vspace("20px")`

### Model structure

The log-transformed number of gene copies per liter at catchment area $i=1, \ldots, 303$ and week $t=1,\ldots, 44$ is modelled as:

$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$

with $\sigma^2_y$ as the measurement error variance. For the latent mean concentration $\mu_{it}$ we specify a linear model:

$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  {\color{red}u_i} + v_t + z_{it}$$

- $\color{red}{u_i}$: spatial random effect with an exchangeable prior 

---


count: false

# Statistical model

.content-box-green[The model should 
- quantify the relationship between the weekly viral concentration in wastewater samples at the STW catchment area level and the set of covariates 
- predict weekly viral concentration at any chosen spatial resolution, together with the associated predictive uncertainty.]

`r vspace("20px")`

### Model structure

The log-transformed number of gene copies per liter at catchment area $i=1, \ldots, 303$ and week $t=1,\ldots, 44$ is modelled as:

$$y_{it} \sim \text{Normal}(\mu_{it}, \sigma^2_y)$$

with $\sigma^2_y$ as the measurement error variance. For the latent mean concentration $\mu_{it}$ we specify a linear model:

$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  u_i + {\color{red}v_t} + z_{it}$$

- $\color{red}{v_t}$: temporal random effect modelled with first order random walk

---

# Spatio-temporal random effect

$$\mu_{it} = \alpha_{region_{g_i}} + \bm{x}_i \bm{\beta} +  u_i + v_t + {\color{red}z_{it}}$$

- $\color{red}{z_{it}}$: spatio-temporal random effect, modelled as a temporal autoregressive structure with spatially correlated innovations, similar to `r Citet(bibfile, "cameletti2011comparing")`:
\begin{align*}
t = 1 \qquad \boldsymbol{z}_t &\sim \text{Normal}(0,\Sigma_{z})\\
t > 1 \qquad \boldsymbol{z}_t &\sim \text{Normal}(\rho\boldsymbol{z}_{t-1},\Sigma_{z})
\end{align*}
with $\rho$ as the temporal autoregressive coefficient. 

--

The  $\Sigma_{z}$ matrix represents the spatial structure and is defined using a Mat&eacute;rn covariance function.

$$\text{Cov}(\bm{z}_i,\bm{z}_s)\ = \frac{\sigma^2_{z}}{\Gamma(\lambda)2^{\lambda-1}}(\kappa ||\bm{z}_i - \bm{z}_s||)^{\lambda}K_{\lambda}(\kappa ||\bm{z}_i - \bm{z}_s||)$$

where $\Gamma(\lambda)$ is the gamma function, $K_{\lambda}$ is the modified Bessel function of second kind with order $\lambda$. The parameter $\kappa>0$ controls the range $\rho=\sqrt{8\lambda}/\kappa$ and indicate the distance where the spatial correlation is close to 0.1. 

---

# Spatial prediction

- We consider the population-weighted centroids of all the 32,844 Lower Super Output Areas (LSOAs) in England, these being the smallest areas 
for which all the covariates are available. 

Given observed data $\boldsymbol{y}$ we sample $\boldsymbol{y}^\prime$, the viral concentrations at the LSOA centroids over 44 weeks, from the posterior predictive distribution:
\begin{align}
\pi(\boldsymbol{y}^\prime \mid \boldsymbol{y}) &= \int p(\boldsymbol{y}^\prime \mid \boldsymbol{\theta}) p(\boldsymbol{\theta} \mid \boldsymbol{y}) d\boldsymbol{\theta}\\
\boldsymbol{\theta} &= (\alpha, \boldsymbol{\beta}, \boldsymbol{v}, \boldsymbol{g}, \boldsymbol{u}, \boldsymbol{z}, \sigma^2_y)\ .
\end{align}


--

`r vspace("30px")`


The posterior predictive distribution at the LSOA centroids can be combined to return corresponding predictions at coarser geographical scales, as required. 

  - Here we consider all the Lower Tier Local Authorities in England and average the corresponding LSOA values, weighted by the population in each LSOA.


- Implemented in R-INLA 

---


# Detection of areas at increased risk of outbreaks

.panelset[
.panel[.panel-name[Detection]
We used the posterior predictive distribution to detect increases in the wastewater viral concentration. For each LTLA $(l=1,\ldots,L)$ an increase in viral concentration over two consecutive weeks is the event (E):
\begin{eqnarray}
E = I(\mu_{lt} > \mu_{l(t-1)} > \mu_{l(t-2)})
\end{eqnarray}
where $I(\cdot)$ is the indicator function.

`r vspace("20px")`
A high predictive probability, ${\rm Prob}(E)>c$ for a pre-specified threshold $c$, would then trigger a warning of a potential outbreak. This approach has been used for public health surveillance in both high-income `r Citep(bibfile,"diggle2005env")` and low-to-middle-income `r Citep(bibfile,"diggle2007anntrop")` countries. 
]

.panel[.panel-name[Mapping]

`r include_fig("Detection.png",width="50%")`
]
]

---

#Interactive dashboard
To visualise these rich model outputs, we have created a dynamic and interactive dashboard

`r vspace("-50px")`

```{r, echo=FALSE, out.width="140%", out.extra='style="transform: scale(.8);border:none"'}
options(htmltools.dir.version = FALSE, htmltools.preserve.raw = FALSE)
knitr::include_url("https://b-rowlingson.gitlab.io/wwatlas/", height="100%")
```

---

# Wastewater viral concentration and COVID-19 prevalence

.pull-left[The relationship between viral concentration in
wastewater and COVID-19 prevalence appears to be .red[complex
and varying over both space and time.]]


.pull-right[`r include_fig("fig6.jpg", width="50%")`]

---

# Wastewater in the post-pandemic era to monitor COVID

- Data on randomised surveys and testing become more sparse
`r vspace("-5px")`
$\rightarrow$ Provide information on coarse geographical resolution (e.g. regional or national)

`r vspace("20px")`

- Wastewater can track prevalence well and is cheaper to measure
`r vspace("-5px")`
$\rightarrow$ A monitoring system needs to be maintained 


--

`r vspace("20px")`

What data do we have (in the context of England)?

<!--a. Assuming we have wastewater viral concentrations regularly measured at STWs across the country

$\rightarrow$ we can use the geostatistical model to predict the LTLA weekly concentration.

b. .red[LTLA weekly prevalence] estimates are only available up to a certain time point, but estimates at the weekly-coarse level remain available -->


`r include_fig("WWCovid.png", width="60%")`

.content-box-green[Can wastewater data help disaggregate coarse level prevalence to
a finer spatial scale?
]

---

# Modelling framework

- Coarse (national) and local (LTLA) models jointly fitted

- Wastewater predictions available from the geostatistical model

- Posterior distribution of COVID-19 prevalence obtained through `r Citet(bibfile,"nicholson2022improving")` 

--

### Coarse (national)
\begin{align*}
\tilde{P}_{t} &\sim \mbox{N}(\tilde{\mu}_{t},\tilde{\sigma}^2_{t})\\
\tilde{\mu}_{t} &= \alpha + b_t + d_t \tilde{y}^{\prime}_{t}
\end{align*}

where 

- $\tilde{P}_{t}$ and $\tilde{\sigma}^2_{t}$ are posterior mean and variance of the national aggregation of the prevalence (population weighted)
- $\tilde{y}^{\prime}_{t}$ is a similar aggregation for wastewater concentration

`r vspace("20px")`

- $b_t$ is modelled using a RW1
- $d_t$ is modelled using a RW2

---

# Modelling framework

- Coarse (national) and local (LTLA) models jointly fitted

- Wastewater predictions available from the geostatistical model

- Posterior distribution of COVID-19 prevalence obtained through `r Citet(bibfile,"nicholson2022improving")` 

--

### Local
\begin{align*}
p_{lt} &\sim \mbox{N}(\mu_{lt},\sigma^2_{lt})\\
\mu_{lt} &= (\alpha + u_l) + (b_t + v_{lt}) + (d_t + m_l)y^{\prime}_{lt}\\
\bm{y}^{\prime}_{t} &\sim \mbox{MVN}(\bm{g}_t, \bm{S}_t)
\end{align*}

where 

- $p_{lt}$ and $\sigma^2_{lt}$ are posterior mean and variance of the COVID-19 prevalence 
- $\bm{g}_t$ and $\bm{S}_t$ are the posterior mean and covariance matrix from the wastewater model
- $u_{l}$, $v_{lt}, m_l$ are modelled as exchangeable


---

# Implementation

- Both the local and national components are jointly fitted

- Alternative models: 

1\. No wastewater

`r vspace("10px")`
$\rightarrow$ temporal trend for the coarse level model $\tilde{\mu}_{t} = \alpha + b_t + d_t$
`r vspace("10px")`
$\rightarrow$ spatial and temporal trend for the local level model $\mu_{lt} = (\alpha + u_l) + (b_t + v_{lt})$

`r vspace("30px")`

2\. No temporal varying coefficients
`r vspace("10px")`
$\rightarrow$ linear relationship with wastewater $\tilde{\mu}_{t} = \alpha + b_t + d \tilde{y}^{\prime}_{t}$
`r vspace("10px")`
$\rightarrow$ spatial-varying coefficient for wastewater $\mu_{lt} = (\alpha + u_l) + (b_t + v_{lt}) + (d + m_l)y^{\prime}_{lt}$

--

`r vspace("30px")`

- *Study period*: 40 weeks

- **Training window**: first 20 weeks (.red[LTLA + national weekly prevalence] + .blue[LTLA + national weekly wastewater])

- **Disaggregation window**: subsequent 5/10/15/20 weeks (.red[national weekly prevalence] + .blue[LTLA + national weekly wastewater])

---

# Results

.pull-left[`r include_fig("Results.png", width="150%")`
]

.pull-right[
- Using wastewater data helps to improve the quality of the disaggregated LTLA prevalence (lower RMSE and better coverage) consistently across all disaggregation horizons.

- Not large difference between the model 2 and model 3

- Some geographical variation

]

`r vspace("-20px")`
`r include_fig("Boxplot.png", width="80%")`


---

# Results

.pull-left[`r vspace("40px")`
`r include_fig("Heatmap.png", width="100%")`
]


.pull-right[`r vspace("5px")`
`r include_fig("Scatterplot.png", width="80%")`
]

$\rightarrow$ Some periods (e.g. Christmas are showing larger differences)

$\rightarrow$ Some areas are consistently underestimated/overestimated

---

# What's next

How can wastewater be incorporated into a real-time, multiplex public health surveillance system?

`r vspace("20px")`

`r include_fig("Surveillance_representation.png", width="60%")`

---

name: COPD

`r vspace("250px")`

.myblue[.center[.huge[
**Space-Time modelling: Temperature and Health**]]]



---

# COPD and temperature
.panelset[
.panel[.panel-name[Temperature and Health]

.pull-left[
- Typically U-shaped relationship between temperature and health.
- Cold, dry air or hot air can trigger a flare-up.
- Different confounding, different lags across different temperatures.
]

.pull-right[

`r include_fig("TemperatureHealth.png")`
]

]

.panel[.panel-name[COPD]

`r include_fig("COPD_lung.png",width="60%")`

- In England, 115,000 emergency admissions and 24,000 deaths per year.
- COPD exacerbations: Bacteria, viruses and air-pollution.
- The role of temperature is unclear. 

]
]

---

# Aim

.content-box-green[Examine the effect of warm temperatures on COPD hospitalizations.]

`r vspace("30px")`

Secondary objectives:
`r vspace("20px")`
- Examine effect modification by age, sex and space.
`r vspace("20px")`
- Assess how contextual characteristics, such as green space, average temperature, deprivation and urbanicity affect population vulnerability. 
`r vspace("20px")`
- Calculate the number of COPD cases attributable to temperature.

---

# Outcome and exposure

.pull-left[
- COPD hospitalization (ICD10 J40-44) 2007-2018 (NHS digital).
- Individual data/ 100m grid spatial resolution.
- Summer months.

--

- Daily maximum temperature 2007-2018 at 1km grid from MetOffice.
- lag0-2.
]

.pull-right[
`r include_fig("temperatureLTLA.png")`
]

---

# Case-crossover design

- Epidemiological study design for transient environmental exposures
`r vspace("20px")`    
- Sample from same month and days
`r vspace("20px")`    
- Exposure period 0-2 days before the hospital episode (mean temperature)

`r include_fig("FigCaseCrossOver.png", width="60%")`

---

# Confouders and spatial effect modifiers

.panel-set[.panel[
panel-name[Confounders]
```{r conf-tables}
Confounder<- c("PM2.5","O3","Relative Humidity","Holidays")
Source <- c("MetOffice","MetOffice","MetOffice","ONS")
Space<-c("1Km","1Km","10Km", "nationwide")
Time<- rep("daily",4)
Year<-rep("2007-2018",4)

df<- data.frame(Confounder,Source,Space,Time,Year)
knitr::kable(df)
```
]

.panel[panel-name[Effect modifiers]

`r include_fig("SpatEffMod.png")`
]
]

---


# Acknowledgments
.pull-left[
.red[Source apportionment]:

`r include_fig("MRClogo.png", width="60%")`

**Monica Pirani**, **Georges Bucyibaruta** (ICL, EBS)

**Gary Fuller**, **Anya Tremper**, **Dave Green** (IC-ERG)

**Christina Mitsakou** (UKHSA)

]

.pull-right[
.red[Wastewater based epidemiology]:

`r include_fig("Turing.png", width="60%")`

**Guangquan Li** (Northumbria University)

**Radka Jersakova**, **Callum Mole**, **Camila Rangel Smith**  (Alan Turing Institute)

**Barry Rowlingson**, **Peter Diggle** (Lancaster University)

**Sylvia Richardson** (MRC Biostatistics Unit, Cambridge)

**George Nicholson**, **Chris Holmes** (Oxford University)

**Fatemeh Torabi** (Swansea University)

**Matt Wade**, **Hubert Denise**, **William Rowe**, **Daniel James**, **Jasmine Grimsley** (UKHSA)

]

---